Index: shake.spec
===================================================================
--- shake.spec	(revision 1)
+++ shake.spec	(working copy)
@@ -25,6 +25,7 @@
 License:        BSD-3-Clause
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
+Patch0:         shake-0.19.1.patch
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-binary-devel
 BuildRequires:  ghc-bytestring-devel
@@ -88,7 +89,7 @@
 This package provides the Haskell %{name} library development files.
 
 %prep
-%autosetup
+%autosetup -p1
 
 %build
 %ghc_lib_build
