Index: hledger.spec
===================================================================
--- hledger.spec	(revision 3)
+++ hledger.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        1.1
 Release:        0
 Summary:        Command-line interface for the hledger accounting tool
-License:        GPL-1.0+
+License:        GPL-3.0+
 Group:          Development/Languages/Other
 Url:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz

