Index: ghc-mod.spec
===================================================================
--- ghc-mod.spec	(revision 2)
+++ ghc-mod.spec	(working copy)
@@ -64,6 +64,10 @@
 BuildRequires:  ghc-time-devel
 BuildRequires:  ghc-transformers-base-devel
 BuildRequires:  ghc-transformers-devel
+Requires:       ghc-cabal-helper
+Requires:       ghc
+Suggests:       cabal-install
 Suggests:       emacs-haskell-mode
+Suggests:       hlint
 %if %{with tests}
 BuildRequires:  ghc-doctest-devel

