Index: ghc-mod.spec
===================================================================
--- ghc-mod.spec	(revision 4)
+++ ghc-mod.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        5.5.0.0
 Release:        0
 Summary:        Happy Haskell Programming
-License:        AGPL-3.0
+License:        AGPL-3.0+ and BSD-3-Clause
 Group:          Development/Languages/Other
 Url:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
@@ -61,6 +61,7 @@
 BuildRequires:  ghc-time-devel
 BuildRequires:  ghc-transformers-base-devel
 BuildRequires:  ghc-transformers-devel
+Suggests:       emacs-haskel-mode
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 %if %{with tests}
 BuildRequires:  ghc-doctest-devel
@@ -102,10 +103,19 @@
 
 %build
+%define cabal_configure_options --libexecdir=%{_libexecdir}
 %ghc_lib_build
 
 %install
 %ghc_lib_install
+%ghc_fix_dynamic_rpath %{pkg_name} ghc-modi
+mkdir -p %{buildroot}%{_sysconfdir}/profile.d
+touch %{buildroot}%{_sysconfdir}/profile.d/ghc-mod.sh
+touch %{buildroot}%{_sysconfdir}/profile.d/ghc-mod.csh
+echo 'export cabal_helper_libexecdir=%{_libexecdir}' > %{buildroot}%{_sysconfdir}/profile.d/ghc-mod.sh
+echo 'setenv cabal_helper_libexecdir %{_libexecdir}' > %{buildroot}%{_sysconfdir}/profile.d/ghc-mod.csh
-%ghc_fix_dynamic_rpath %{pkg_name} ghc-modi
+rm %{buildroot}%{_datadir}/%{name}-%{version}/LICENSE
+rm %{buildroot}%{_datadir}/%{name}-%{version}/COPYING.AGPL3
+rm %{buildroot}%{_datadir}/%{name}-%{version}/COPYING.BSD3
 
 %check
 %if %{with tests}
@@ -125,26 +134,17 @@
 
 %files
 %defattr(-,root,root,-)
-%doc COPYING.BSD3
-%doc COPYING.AGPL3
-%doc ChangeLog
+%doc LICENSE COPYING.AGPL3 COPYING.BSD3
 %{_bindir}/%{name}
 %{_bindir}/ghc-modi
-%dir %{_datadir}/%{name}-%{version}
-%dir %{_datadir}/%{name}-%{version}/elisp
-%{_datadir}/%{name}-%{version}/elisp/Makefile
-%{_datadir}/%{name}-%{version}/elisp/*.el
-%{_datadir}/%{name}-%{version}/LICENSE
-%{_datadir}/%{name}-%{version}/COPYING.BSD3
-%{_datadir}/%{name}-%{version}/COPYING.AGPL3
+%{_datadir}/%{name}-%{version}
+%config %{_sysconfdir}/profile.d/ghc-mod.sh
+%config %{_sysconfdir}/profile.d/ghc-mod.csh
 
 %files -n ghc-%{name} -f ghc-%{name}.files
 %defattr(-,root,root,-)
-%doc COPYING.BSD3
-%doc COPYING.AGPL3
 
 %files -n ghc-%{name}-devel -f ghc-%{name}-devel.files
 %defattr(-,root,root,-)
-%doc ChangeLog
 
 %changelog

