Index: ghc-avers-api.spec
===================================================================
--- ghc-avers-api.spec	(revision 1)
+++ ghc-avers-api.spec	(working copy)
@@ -39,7 +39,7 @@
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 
 %description
-See README.
+Types describing the core and extended Avers APIs.
 
 %package devel
 Summary:        Haskell %{pkg_name} library development files

