Index: ghc-opml-conduit.spec
===================================================================
--- ghc-opml-conduit.spec	(revision 2)
+++ ghc-opml-conduit.spec	(working copy)
@@ -22,6 +22,6 @@
 Version:        0.6.0.1
 Release:        0
 Summary:        Streaming parser/renderer for the OPML 2.0 format
-License:        SUSE-Public-Domain
+License:        WTFPL
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

