Index: ghc-lambdacube-gl.spec
===================================================================
--- ghc-lambdacube-gl.spec	(revision 10)
+++ ghc-lambdacube-gl.spec	(working copy)
@@ -25,7 +25,6 @@
 Group:          Development/Languages/Other
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
-BuildRequires:  chrpath
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-GLFW-b-devel
 BuildRequires:  ghc-JuicyPixels-devel
@@ -66,10 +65,9 @@
 
 %build
-%define cabal_configure_options -ftestclient -fexample
+%define cabal_configure_options -f-testclient -f-example
 %ghc_lib_build
 
 %install
 %ghc_lib_install
-%{ghc_fix_rpath} %{pkg_name}-%{version}
 
 %post devel
@@ -84,9 +82,6 @@
 %files -f %{name}.files
 %defattr(-,root,root,-)
 %doc LICENSE
-%{_bindir}/lambdacube-gl-hello
-%{_bindir}/lambdacube-gl-hello-obj
-%{_bindir}/lambdacube-gl-test-client
 
 %files devel -f %{name}-devel.files
 %defattr(-,root,root,-)

