Index: ghc-gogol-admin-emailmigration.spec
===================================================================
--- ghc-gogol-admin-emailmigration.spec	(revision 5)
+++ ghc-gogol-admin-emailmigration.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.1.0
 Release:        0
 Summary:        Google Email Migration API v2 SDK
-License:        Unknown
+License:        MPL-2.0
 Group:          Development/Languages/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

