Index: ghc-SDL-mixer.spec
===================================================================
--- ghc-SDL-mixer.spec	(revision 2)
+++ ghc-SDL-mixer.spec	(working copy)
@@ -32,6 +32,8 @@
 BuildRequires:  ghc-rpm-macros
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 # End cabal-rpm deps
+BuildRequires:  pkgconfig
+BuildRequires:  pkgconfig(SDL_mixer)
 
 %description
 SDL_mixer is a sample multi-channel audio mixer library. It supports any number

