Index: ghc-Decimal.spec
===================================================================
--- ghc-Decimal.spec	(revision 2)
+++ ghc-Decimal.spec	(working copy)
@@ -25,6 +25,7 @@
 License:        BSD-3-Clause
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
+Patch1:         https://gitlab.haskell.org/ghc/head.hackage/-/raw/master/patches/Decimal-0.5.1.patch
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-deepseq-devel
 BuildRequires:  ghc-rpm-macros
@@ -51,7 +52,7 @@
 This package provides the Haskell %{pkg_name} library development files.
 
 %prep
-%autosetup -n %{pkg_name}-%{version}
+%autosetup -n %{pkg_name}-%{version} -p1
 
 %build
 %ghc_lib_build
