Index: ghc-rbpcp-api.spec
===================================================================
--- ghc-rbpcp-api.spec	(revision 1)
+++ ghc-rbpcp-api.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.1.0.0
 Release:        0
 Summary:        RESTful Bitcoin Payment Channel Protocol Servant API description
-License:        Unknown
+License:        CC0-1.0
 Group:          Development/Languages/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

