Index: ghc-monadLib.spec
===================================================================
--- ghc-monadLib.spec (revision 2)
+++ ghc-monadLib.spec (revision 3)
@@ -21,7 +21,7 @@
 Version:        3.7.3
 Release:        0
 Summary:        A collection of monad transformers
-License:        BSD-3-Clause
+License:        MIT
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

