Index: idris.spec
===================================================================
--- idris.spec (revision 17)
+++ idris.spec (revision 18)
@@ -26,5 +26,6 @@
 Group:          Development/Languages/Other
 Url:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
+Source10:       idris-rpmlintrc
 BuildRequires:  chrpath
 BuildRequires:  ghc-Cabal-devel
