Index: idris.spec
===================================================================
--- idris.spec	(revision 22)
+++ idris.spec	(working copy)
@@ -131,6 +131,10 @@
 %ghc_lib_install
 %ghc_fix_rpath %{pkg_name}-%{version}
 
+# Fix unnecessary executable permission bits on header and library files.
+find "%{buildroot}%{_libexecdir}/%{name}-%{version}" -name "*.a" -exec chmod -x {} +
+find "%{buildroot}%{_libexecdir}/%{name}-%{version}" -name "*.h" -exec chmod -x {} +
+
 %check
 %cabal_test
 

