Index: idris.spec
===================================================================
--- idris.spec	(revision 20)
+++ idris.spec	(working copy)
@@ -26,6 +26,7 @@
 Group:          Development/Languages/Haskell
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
+Source10:       idris-rpmlintrc
 BuildRequires:  chrpath
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-aeson-devel

