Index: ghc-io-streams-haproxy.spec
===================================================================
--- ghc-io-streams-haproxy.spec	(revision 5)
+++ ghc-io-streams-haproxy.spec	(working copy)
@@ -62,6 +62,7 @@
 %autosetup -n %{pkg_name}-%{version}
 cp -p %{SOURCE1} %{pkg_name}.cabal
 cabal-tweak-dep-ver 'attoparsec' '< 0.14' '< 0.15'
+cabal-tweak-dep-ver 'base' '< 4.15' '< 4.16'
 
 %build
 %ghc_lib_build
