Index: ghc-hxt-regex-xmlschema.spec
===================================================================
--- ghc-hxt-regex-xmlschema.spec	(revision 1)
+++ ghc-hxt-regex-xmlschema.spec	(working copy)
@@ -26,6 +26,7 @@
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
 Source1:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/revision/1.cabal#/%{pkg_name}.cabal
+Patch:          hxt-regex-xmlschema-9.2.0.3.patch
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-bytestring-devel
 BuildRequires:  ghc-hxt-charproperties-devel
@@ -64,7 +65,7 @@
 files.
 
 %prep
-%autosetup -n %{pkg_name}-%{version}
+%autosetup -p1 -n %{pkg_name}-%{version}
 cp -p %{SOURCE1} %{pkg_name}.cabal
 
 %build
