Index: ghc-typography-geometry.spec
===================================================================
--- ghc-typography-geometry.spec	(revision 1)
+++ ghc-typography-geometry.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        1.0.0.1
 Release:        0
 Summary:        Drawings for printed text documents
-License:        GPL-1.0+
+License:        GPL-2.0+
 Group:          Development/Languages/Haskell
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

