Index: pandoc-cli.spec
===================================================================
--- pandoc-cli.spec	(revision 1)
+++ pandoc-cli.spec	(working copy)
@@ -34,6 +34,8 @@
 BuildRequires:  ghc-wai-extra-devel
 BuildRequires:  ghc-warp-devel
 ExcludeArch:    %{ix86}
+Provides:       pandoc = 3.0.1
+Obsoletes:      pandoc < 3
 
 %description
 Pandoc-cli provides a command-line executable that uses the pandoc library to
