Index: ghc-asn1-types.spec
===================================================================
--- ghc-asn1-types.spec	(revision 1)
+++ ghc-asn1-types.spec	(working copy)
@@ -44,6 +44,7 @@
 Requires:       ghc-compiler = %{ghc_version}
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
+Provides:       %{name}-static = %{version}-%{release}
 
 %description devel
 This package provides the Haskell %{pkg_name} library development files.

