Index: ghc-mod.spec
===================================================================
--- ghc-mod.spec	(revision 16)
+++ ghc-mod.spec	(working copy)
@@ -116,6 +116,9 @@
 mkdir -p %{buildroot}%{_sysconfdir}/profile.d
 echo 'export cabal_helper_libexecdir=%{_libexecdir}/%{_arch}-%{_os}-ghc-%{ghc_version}/cabal-helper' > %{buildroot}%{_sysconfdir}/profile.d/ghc-mod.sh
 echo 'setenv cabal_helper_libexecdir %{_libexecdir}/%{_arch}-%{_os}-ghc-%{ghc_version}/cabal-helper' > %{buildroot}%{_sysconfdir}/profile.d/ghc-mod.csh
+rm %{buildroot}%{_datadir}/%{name}-%{version}/LICENSE
+rm %{buildroot}%{_datadir}/%{name}-%{version}/COPYING.AGPL3
+rm %{buildroot}%{_datadir}/%{name}-%{version}/COPYING.BSD3
 
 %check
 %cabal_test

