From 2e99cbe21b6dd2b6cfdeb6f0c7e4b75ce1dae98a Mon Sep 17 00:00:00 2001
From: Peter Simons <simons@cryp.to>
Date: Wed, 20 Jul 2016 23:46:40 +0200
Subject: [PATCH 3/5] improve group tag

---
 pandoc.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pandoc.spec b/pandoc.spec
index e097434..dbe6fc7 100644
--- a/pandoc.spec
+++ b/pandoc.spec
@@ -23,7 +23,7 @@ Version:        1.17.1
 Release:        0
 Summary:        Conversion between markup formats
 License:        GPL-2.0+
-Group:          Development/Languages/Haskell
+Group:          Productivity/Publishing/Other
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
 Source1:        https://hackage.haskell.org/package/%{name}-%{version}/revision/1.cabal
-- 
2.9.0

