From 31ba9db40d050b679aef17f56ce9d7d32843f054 Mon Sep 17 00:00:00 2001
From: Peter Simons <simons@cryp.to>
Date: Wed, 20 Jul 2016 23:46:23 +0200
Subject: [PATCH 2/5] suggest installing texlive-latex-bin-bin

---
 pandoc.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pandoc.spec b/pandoc.spec
index dd39a7a..e097434 100644
--- a/pandoc.spec
+++ b/pandoc.spec
@@ -75,6 +75,7 @@ BuildRequires:  ghc-xml-devel
 BuildRequires:  ghc-yaml-devel
 BuildRequires:  ghc-zip-archive-devel
 BuildRequires:  ghc-zlib-devel
+Suggests:       texlive-latex-bin-bin
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 %if %{with tests}
 BuildRequires:  ghc-Diff-devel
-- 
2.9.0

