Index: pandoc.spec
===================================================================
--- pandoc.spec	(revision a1a12ec3d59656fc410f90507c39652f)
+++ pandoc.spec	(working copy)
@@ -26,7 +26,8 @@
 Group:          Productivity/Publishing/Other
 Url:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
 Source1:        https://hackage.haskell.org/package/%{name}-%{version}/revision/5.cabal#/%{name}.cabal
+Source10:       pandoc-pdf.readme
 BuildRequires:  chrpath
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-HTTP-devel
@@ -106,6 +107,19 @@
 writers, which convert this native representation into a target format.
 Thus, adding an input or output format requires only adding a reader or writer.
 
+%if 0%{?is_opensuse}
+%package pdf
+Summary:        Meta package for support generating pdf with pandoc
+Group:          Productivity/Publishing/PDF
+Requires:       %{name} = %{version}
+Requires:       texlive-latex-bin-bin
+# bsc#1014590
+Requires:       texlive-upquote
+
+%description pdf
+Meta package for support generating pdf with pandoc.
+%endif
+
 %package -n ghc-%{name}
 Summary:        Haskell %{name} library
 Group:          System/Libraries
@@ -126,7 +140,8 @@
 
 %prep
 %setup -q
 cp -p %{SOURCE1} %{name}.cabal
+cp %{SOURCE10} .
 
 %build
 %ghc_lib_build
@@ -227,6 +242,12 @@
 %{_datadir}/%{name}-%{version}/MANUAL.txt
 %{_datadir}/%{name}-%{version}/COPYRIGHT
 
+%if 0%{?is_opensuse}
+%files pdf
+%defattr(-,root,root,-)
+%doc pandoc-pdf.readme
+%endif
+
 %files -n ghc-%{name} -f ghc-%{name}.files
 %defattr(-,root,root,-)
 %doc COPYING.md

