Index: pandoc.spec
===================================================================
--- pandoc.spec	(revision 72)
+++ pandoc.spec	(working copy)
@@ -28,6 +28,7 @@
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-Glob-devel
+BuildRequires:  fdupes
 BuildRequires:  ghc-JuicyPixels-devel
 BuildRequires:  ghc-SHA-devel
 BuildRequires:  ghc-aeson-devel
@@ -149,6 +150,8 @@
 
 %install
 %ghc_lib_install
+# Link duplicate template files
+%fdupes %{buildroot}%{_datadir}/%{pkg_name}-%{version}/data/templates/
 
 %check
 %cabal_test
