Index: ghc-hidapi.spec
===================================================================
--- ghc-hidapi.spec	(revision 1)
+++ ghc-hidapi.spec	(working copy)
@@ -33,6 +33,7 @@
 BuildRequires:  ghc-rpm-macros
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 # End cabal-rpm deps
+BuildRequires:  libudev-devel
 
 %description
 Haskell bindings to the HIDAPI library (<http://www.signal11.us/oss/hidapi/>).

