Index: ghc-terminfo.spec
===================================================================
--- ghc-terminfo.spec	(revision 10)
+++ ghc-terminfo.spec	(working copy)
@@ -28,5 +28,6 @@
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-rpm-macros
+BuildRequires:  ncurses-devel
 
 %description
 This library provides an interface to the terminfo database (via bindings to
@@ -42,6 +43,7 @@
 Requires:       ghc-compiler = %{ghc_version}
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
+Requires:       ncurses-devel
 
 %description devel
 This package provides the Haskell %{pkg_name} library development files.

