Index: ghc-terminfo.spec
===================================================================
--- ghc-terminfo.spec	(revision 5)
+++ ghc-terminfo.spec	(working copy)
@@ -44,6 +44,8 @@
 Requires:       ghc-compiler = %{ghc_version}
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
+Requires:       pkgconfig
+Requires:       pkgconfig(ncurses)
 
 %description devel
 This package provides the Haskell %{pkg_name} library development files.

