Index: ghc-MonadRandom.spec
===================================================================
--- ghc-MonadRandom.spec	(revision 3)
+++ ghc-MonadRandom.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.4.2.3
 Release:        0
 Summary:        Random-number generation monad
-License:        Unknown
+License:        MIT
 Group:          Development/Languages/Other
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

