Index: ghc-wai-middleware-consul.spec
===================================================================
--- ghc-wai-middleware-consul.spec	(revision 1)
+++ ghc-wai-middleware-consul.spec	(working copy)
@@ -49,5 +49,8 @@
 BuildRequires:  ghc-wai-devel
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+BuildRequires:  ghc-wai-app-static-devel
+BuildRequires:  ghc-wai-extra-devel
+BuildRequires:  ghc-warp-devel
 
 %description
 Proxies data to/from Consul. Watches for updates.

