Index: ghc-amazonka-opsworks.spec
===================================================================
--- ghc-amazonka-opsworks.spec	(revision 5)
+++ ghc-amazonka-opsworks.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        1.4.3
 Release:        0
 Summary:        Amazon OpsWorks SDK
-License:        Unknown
+License:        MPL-2.0
 Group:          Development/Languages/Haskell
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

