Index: pandoc.spec
===================================================================
--- pandoc.spec	(revision 18)
+++ pandoc.spec	(working copy)
@@ -155,6 +155,7 @@
 %ghc_fix_rpath %{pkg_name}-%{version}
 # Link duplicate template files
 %fdupes %{buildroot}%{_datadir}/%{pkg_name}-%{version}/data/templates/
+install -D -m444 man/pandoc.1 %{buildroot}%{_mandir}/man1/pandoc.1
 
 %check
 %cabal_test

