Index: ghc-riak.spec
===================================================================
--- ghc-riak.spec	(revision 5)
+++ ghc-riak.spec	(working copy)
@@ -22,6 +22,6 @@
 Version:        1.0.1.1
 Release:        0
 Summary:        A Haskell client for the Riak decentralized data store
-License:        Unknown
+License:        Apache-2.0
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

