Index: ghc-hackage-security.spec
===================================================================
--- ghc-hackage-security.spec	(revision 1)
+++ ghc-hackage-security.spec	(working copy)
@@ -27,6 +27,7 @@
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
 Source1:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/revision/6.cabal#/%{pkg_name}.cabal
+Patch1:         https://raw.githubusercontent.com/hvr/head.hackage/master/patches/hackage-security-0.5.3.0.patch#/fix-ghc-8.8.x-build.patch
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-base16-bytestring-devel
 BuildRequires:  ghc-base64-bytestring-devel
@@ -87,6 +88,8 @@
 %prep
 %setup -q -n %{pkg_name}-%{version}
 cp -p %{SOURCE1} %{pkg_name}.cabal
+%patch01 -p1
+cabal-tweak-dep-ver base '< 4.13' '< 5'
 
 %build
 %ghc_lib_build
