Index: ghc-hackage-security.spec
===================================================================
--- ghc-hackage-security.spec	(revision 7)
+++ ghc-hackage-security.spec	(working copy)
@@ -53,6 +53,7 @@
 BuildRequires:  ghc-tasty-quickcheck-devel
 BuildRequires:  ghc-temporary-devel
 %endif
+Patch01:        https://raw.githubusercontent.com/hvr/head.hackage/master/patches/hackage-security-0.5.3.0.patch#/fix-ghc-8.8.x-build.patch
 
 %description
 The hackage security library provides both server and client utilities for
@@ -85,6 +86,9 @@
 %prep
 %setup -q -n %{pkg_name}-%{version}
 cp -p %{SOURCE1} %{pkg_name}.cabal
+%patch01 -p1
+cabal-tweak-dep-ver base '< 4.13' '< 5'
+cabal-tweak-dep-ver network-uri '< 2.7' '< 3'
 
 %build
 %ghc_lib_build
