Index: ghc-yesod-auth-oauth2.spec
===================================================================
--- ghc-yesod-auth-oauth2.spec	(revision 3)
+++ ghc-yesod-auth-oauth2.spec	(working copy)
@@ -50,5 +50,7 @@
 BuildRequires:  ghc-hspec-devel
 %endif
+BuildRequires:  ghc-load-env-devel
+BuildRequires:  ghc-yesod-devel
 
 %description
 Library to authenticate with OAuth 2.0 for Yesod web applications.

