Index: ghc-mysql.spec
===================================================================
--- ghc-mysql.spec	(revision 5)
+++ ghc-mysql.spec	(working copy)
@@ -32,8 +32,9 @@
 BuildRequires:  ghc-rpm-macros
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 %if %{with tests}
 BuildRequires:  ghc-hspec-devel
 %endif
+BuildRequires:  libmysqlclient-devel
 
 %description
 A low-level client library for the MySQL database, implemented as bindings to

