Index: ghc-basement.spec
===================================================================
--- ghc-basement.spec	(revision 1)
+++ ghc-basement.spec	(working copy)
@@ -25,6 +25,7 @@
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
 Source1:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/revision/2.cabal#/%{pkg_name}.cabal
+Patch1:         https://gitlab.haskell.org/ghc/head.hackage/-/raw/master/patches/basement-0.0.11.patch
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-rpm-macros
 
@@ -42,7 +43,7 @@
 This package provides the Haskell %{pkg_name} library development files.
 
 %prep
-%autosetup -n %{pkg_name}-%{version}
+%autosetup -n %{pkg_name}-%{version} -p1
 cp -p %{SOURCE1} %{pkg_name}.cabal
 
 %build
