Index: ghc-bindings-libzip.spec
===================================================================
--- ghc-bindings-libzip.spec	(revision 7)
+++ ghc-bindings-libzip.spec	(working copy)
@@ -60,6 +60,8 @@
 
 %install
 %ghc_lib_install
+# Work around bogus path in libzip's pkgconfig file. [boo#1087756]
+sed -i -e "s|/usr//usr/|/usr/|g" %{buildroot}%{ghclibdir}/package.conf.d/%{pkg_name}*.conf
 
 %post devel
 %ghc_pkg_recache

