Index: ghc-HDBC-sqlite3.spec
===================================================================
--- ghc-HDBC-sqlite3.spec	(revision 2)
+++ ghc-HDBC-sqlite3.spec	(working copy)
@@ -32,6 +32,7 @@
 BuildRequires:  ghc-mtl-devel
 BuildRequires:  ghc-rpm-macros
 BuildRequires:  ghc-utf8-string-devel
+BuildRequires:  sqlite3-devel
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 # End cabal-rpm deps
 

