Index: ghc-HDBC-postgresql.spec
===================================================================
--- ghc-HDBC-postgresql.spec	(revision 2)
+++ ghc-HDBC-postgresql.spec	(working copy)
@@ -36,6 +36,7 @@
 BuildRequires:  ghc-rpm-macros
 BuildRequires:  ghc-time-devel
 BuildRequires:  ghc-utf8-string-devel
+BuildRequires:  postgresql-devel
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 # End cabal-rpm deps
 

