Index: ghc-SDL-image.spec
===================================================================
--- ghc-SDL-image.spec	(revision 2)
+++ ghc-SDL-image.spec	(working copy)
@@ -27,6 +27,8 @@
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
 Source1:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/revision/1.cabal
 BuildRequires:  ghc-Cabal-devel
+BuildRequires:  pkgconfig
+BuildRequires:  pkgconfig(SDL_image)
 # Begin cabal-rpm deps:
 BuildRequires:  ghc-SDL-devel
 BuildRequires:  ghc-rpm-macros
@@ -43,6 +45,8 @@
 Group:          Development/Libraries/Other
 Requires:       %{name} = %{version}-%{release}
 Requires:       ghc-compiler = %{ghc_version}
+Requires:       pkgconfig
+Requires:       pkgconfig(SDL_image)
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
 

