Index: ghc-swagger.spec
===================================================================
--- ghc-swagger.spec	(revision 5)
+++ ghc-swagger.spec	(working copy)
@@ -22,6 +22,6 @@
 Version:        0.2.2
 Release:        0
 Summary:        Implementation of swagger data model
-License:        Unknown
+License:        MPL-2.0
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

