Index: ghc-cabal-helper.spec
===================================================================
--- ghc-cabal-helper.spec	(revision 3)
+++ ghc-cabal-helper.spec	(working copy)
@@ -86,6 +86,7 @@
 cp -p %{SOURCE1} %{pkg_name}.cabal
 
 %build
+%define cabal_configure_options --libexecdir=%{_libexecdir}
 %ghc_lib_build
 
 %install
@@ -103,5 +104,7 @@
 %files -f %{name}.files
 %doc LICENSE
-%{_bindir}/cabal-helper-wrapper
+%dir %{_libexecdir}/%{_arch}-%{_os}-ghc-%{ghc_version}
+%dir %{_libexecdir}/%{_arch}-%{_os}-ghc-%{ghc_version}/%{pkg_name}-%{version}
+%{_libexecdir}/%{_arch}-%{_os}-ghc-%{ghc_version}/%{pkg_name}-%{version}/cabal-helper-wrapper
 
 %files devel -f %{name}-devel.files

