Index: ghc-cabal-helper.spec
===================================================================
--- ghc-cabal-helper.spec	(revision 7)
+++ ghc-cabal-helper.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        0.6.3.1
 Release:        0
 Summary:        Simple interface to some of Cabal's configuration state used by ghc-mod
-License:        AGPL-3.0
+License:        AGPL-3.0+
 Group:          Development/Languages/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

