Index: ghc-concurrent-supply.spec
===================================================================
--- ghc-concurrent-supply.spec	(revision 5)
+++ ghc-concurrent-supply.spec	(working copy)
@@ -22,6 +22,6 @@
 Version:        0.1.8
 Release:        0
 Summary:        A fast concurrent unique identifier supply with a pure API
-License:        BSD-3-Clause
+License:        BSD-2-Clause
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

