Index: ghc-conduit-parse.spec
===================================================================
--- ghc-conduit-parse.spec	(revision 1)
+++ ghc-conduit-parse.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        0.1.2.0
 Release:        0
 Summary:        Parsing framework based on conduit
-License:        SUSE-Public-Domain
+License:        WTFPL
 Group:          Development/Languages/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

