Index: ghc-waitra.spec
===================================================================
--- ghc-waitra.spec	(revision 6)
+++ ghc-waitra.spec	(working copy)
@@ -27,7 +27,6 @@
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
 Source1:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/revision/2.cabal
-BuildRequires:  chrpath
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-aeson-devel
 BuildRequires:  ghc-bytestring-devel
@@ -74,10 +73,9 @@
 
 %build
-%define cabal_configure_options -fexamples
+%define cabal_configure_options -f-examples
 %ghc_lib_build
 
 %install
 %ghc_lib_install
-%ghc_fix_dynamic_rpath waitra-example
 
 %check
@@ -96,7 +94,6 @@
 %files -f %{name}.files
 %defattr(-,root,root,-)
 %doc LICENSE
-%{_bindir}/waitra-example
 
 %files devel -f %{name}-devel.files
 %defattr(-,root,root,-)

