Index: ghc-mysql.spec
===================================================================
--- ghc-mysql.spec	(revision 1)
+++ ghc-mysql.spec	(working copy)
@@ -32,6 +32,7 @@
 BuildRequires:  ghc-rpm-macros
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 # End cabal-rpm deps
+BuildRequires:  libmysqlclient-devel
 
 %description
 A low-level client library for the MySQL database, implemented as bindings to

