Index: ghc-text-manipulate.spec
===================================================================
--- ghc-text-manipulate.spec	(revision 5)
+++ ghc-text-manipulate.spec	(working copy)
@@ -22,6 +22,6 @@
 Version:        0.2.0.1
 Release:        0
 Summary:        Case conversion, word boundary manipulation, and textual subjugation
-License:        Unknown
+License:        MPL-2.0
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

