Index: git-annex.spec
===================================================================
--- git-annex.spec	(revision 9)
+++ git-annex.spec	(working copy)
@@ -20,7 +20,7 @@
 Version:        6.20160511
 Release:        0
 Summary:        Manage files with git, without checking their contents into git
-License:        GPL-3.0+
+License:        GPL-3.0+ AND AGPL-3.0+ AND BSD-2-Clause AND MIT AND GPL-2.0
 Group:          Development/Languages/Haskell
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz

