Index: git-annex.spec
===================================================================
--- git-annex.spec	(revision 110)
+++ git-annex.spec	(working copy)
@@ -23,6 +23,9 @@
 License:        AGPL-3.0-or-later AND GPL-3.0-or-later AND BSD-2-Clause AND MIT AND GPL-2.0-only
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
+BuildRequires:  bash-completion
+BuildRequires:  chrpath
+BuildRequires:  curl
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-DAV-devel
 BuildRequires:  ghc-IfElse-devel
@@ -120,7 +123,21 @@
 BuildRequires:  ghc-yesod-devel
 BuildRequires:  ghc-yesod-form-devel
 BuildRequires:  ghc-yesod-static-devel
 ExcludeArch:    %{ix86}
+BuildRequires:  git-core
+BuildRequires:  gpg2
+BuildRequires:  lsof
+BuildRequires:  rsync
+Requires(post): desktop-file-utils
+Requires(post): hicolor-icon-theme
+Requires(postun): desktop-file-utils
+Requires(postun): hicolor-icon-theme
+Recommends:     curl
+Recommends:     gpg2
+Recommends:     lsof
+Recommends:     rsync
+Recommends:     ssh
+Suggests:       %{name}-bash-completion
 
 %description
 Git-annex allows managing files with git, without checking the file contents
@@ -139,6 +156,15 @@
 transferring them to other computers. The git-annex webapp makes it easy to set
 up and use git-annex this way.
 
+%package bash-completion
+Summary:        Bash completion for git-annex
+Requires:       %{name} = %{version}
+Requires:       bash-completion
+Supplements:    (%{name} and bash-completion)
+
+%description bash-completion
+Optional dependency offering bash completion for git-annex
+
 %prep
 %autosetup
 
@@ -148,10 +174,25 @@
 
 %install
 %ghc_bin_install
+make DESTDIR=%{buildroot} BUILDER=./Setup install-mans install-completions install-desktop
 
 %files
 %license COPYRIGHT
-%doc CHANGELOG NEWS README doc
+%doc CHANGELOG NEWS README
 %{_bindir}/%{name}
+%{_bindir}/%{name}-shell
+%dir %{_datadir}/icons/hicolor
+%dir %{_datadir}/icons/hicolor/16x16
+%dir %{_datadir}/icons/hicolor/16x16/apps
+%dir %{_datadir}/icons/hicolor/scalable
+%dir %{_datadir}/icons/hicolor/scalable/apps
+%{_sysconfdir}/xdg/autostart/git-annex.desktop
+%{_mandir}/man1/git-annex*.1%{?ext_man}
+%{_datadir}/applications/git-annex.desktop
+%{_datadir}/icons/hicolor/16x16/apps/git-annex.png
+%{_datadir}/icons/hicolor/scalable/apps/git-annex.svg
+
+%files bash-completion
+%{_datadir}/bash-completion/completions/git-annex
 
 %changelog
