Index: git-annex.spec
===================================================================
--- git-annex.spec	(revision 10)
+++ git-annex.spec	(working copy)
@@ -21,7 +21,7 @@
 Release:        0
 Summary:        Manage files with git, without checking their contents into git
 License:        GPL-3.0+ and AGPL-3.0+ and BSD-2-Clause and MIT and GPL-2.0
-Group:          System/Libraries
+Group:          Development/Tools/Version Control
 Url:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
 BuildRequires:  ghc-Cabal-devel

