Index: git-annex.spec
===================================================================
--- git-annex.spec	(revision 8)
+++ git-annex.spec	(working copy)
@@ -113,5 +113,21 @@
 BuildRequires:  ghc-yesod-form-devel
 BuildRequires:  ghc-yesod-static-devel
+# Manually added dependencies which the Cabal file does not declare
+BuildRequires:  git-core
+BuildRequires:  rsync
+Requires:       git-core
+BuildRequires:  bash-completion
+BuildRequires:  curl
+Requires(post): desktop-file-utils
+Requires(post): hicolor-icon-theme
+Requires(postun): desktop-file-utils
+Requires(postun): hicolor-icon-theme
+Recommends:     curl
+Recommends:     gpg2
+Recommends:     lsof
+Recommends:     rsync
+Recommends:     ssh
+Suggests:       %{name}-bash-completion
 
 %description
 Git-annex allows managing files with git, without checking the file contents
@@ -132,6 +148,16 @@
 transferring them to other computers. The git-annex webapp makes it easy to set
 up and use git-annex this way.
 
+%package bash-completion
+Summary:        Bash completion for git-annex
+Group:          System/Shells
+Requires:       %{name} = %{version}
+Requires:       bash-completion
+Supplements:    packageand(%{name}:bash-completion)
+
+%description bash-completion
+Optional dependency offering bash completion for git-annex
+
 %prep
 %setup -q
 
@@ -139,12 +165,35 @@
 %build
 %ghc_bin_build
 
 %install
 %ghc_bin_install
+make DESTDIR=%{buildroot} BUILDER=./Setup install-misc install-mans
+
+%post
+%desktop_database_post
+%icon_theme_cache_post
+
+%postun
+%desktop_database_postun
+%icon_theme_cache_postun
 
 %files
-%license COPYRIGHT
-%doc CHANGELOG NEWS README doc
+%license COPYRIGHT
+%doc README
 %{_bindir}/%{name}
+%{_bindir}/%{name}-shell
+%dir %{_datadir}/icons/hicolor
+%dir %{_datadir}/icons/hicolor/16x16
+%dir %{_datadir}/icons/hicolor/16x16/apps
+%dir %{_datadir}/icons/hicolor/scalable
+%dir %{_datadir}/icons/hicolor/scalable/apps
+%{_sysconfdir}/xdg/autostart/git-annex.desktop
+%{_mandir}/man1/git-annex*.1%{ext_man}
+%{_datadir}/applications/git-annex.desktop
+%{_datadir}/icons/hicolor/16x16/apps/git-annex.png
+%{_datadir}/icons/hicolor/scalable/apps/git-annex.svg
+
+%files bash-completion
+%{_datadir}/bash-completion/completions/git-annex
 
 %changelog

