Index: git-annex.spec
===================================================================
--- git-annex.spec	(revision 64)
+++ git-annex.spec	(working copy)
@@ -173,6 +173,9 @@
 %define cabal_configure_options -ftestsuite
 %ghc_bin_build
 
+%check
+%make_build DESTDIR=%{buildroot} BUILDER=./Setup test
+
 %install
 %ghc_bin_install
 make DESTDIR=%{buildroot} BUILDER=./Setup install-mans install-completions install-desktop
