Index: git-annex.spec
===================================================================
--- git-annex.spec	(revision 55)
+++ git-annex.spec	(working copy)
@@ -179,6 +179,9 @@
 %define cabal_configure_options -ftestsuite
 %ghc_bin_build
 
+%check
+make %{?_smp_mflags} DESTDIR=%{buildroot} BUILDER=./Setup test
+
 %install
 %ghc_bin_install
 make DESTDIR=%{buildroot} BUILDER=./Setup install-mans install-completions install-desktop
