Index: git-annex.spec
===================================================================
--- git-annex.spec (revision 2)
+++ git-annex.spec (revision 3)
@@ -23,7 +23,7 @@
 License:        GPL-3.0+ and AGPL-3.0+ and BSD-2-Clause and MIT and GPL-2.0
 Group:          Development/Tools/Version Control
 URL:            https://hackage.haskell.org/package/%{name}
-Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
+Source0:        https://github.com/peti/git-annex/archive/%{version}.tar.gz#/%{name}-%{version}.tar.gz
 BuildRequires:  bash-completion
 BuildRequires:  curl
 BuildRequires:  ghc-Cabal-devel

