Index: git-annex.spec
===================================================================
--- git-annex.spec	(revision 65)
+++ git-annex.spec	(working copy)
@@ -179,6 +179,8 @@
 %install
 %ghc_bin_install
 make DESTDIR=%{buildroot} BUILDER=./Setup install-mans install-completions install-desktop
+rm %{buildroot}%{_datadir}/fish/vendor_completions.d/git-annex.fish
+rm %{buildroot}%{_datadir}/zsh/site-functions/_git-annex
 
 %files
 %license COPYRIGHT
