Index: git-annex.spec
===================================================================
--- git-annex.spec	(revision 56)
+++ git-annex.spec	(working copy)
@@ -185,6 +185,8 @@
 %install
 %ghc_bin_install
 make DESTDIR=%{buildroot} BUILDER=./Setup install-mans install-completions install-desktop
+rm %{buildroot}%{_datadir}/fish/vendor_completions.d/git-annex.fish
+rm %{buildroot}%{_datadir}/zsh/site-functions/_git-annex
 
 %post
 %desktop_database_post
