Index: git-annex.spec
===================================================================
--- git-annex.spec	(revision 7)
+++ git-annex.spec	(working copy)
@@ -287,6 +287,8 @@
 %{_bindir}/%{name}
 %{_bindir}/%{name}-shell
 %{_bindir}/git-remote-tor-annex
+%{_bindir}/git-remote-annex
+%{_bindir}/git-remote-p2p-annex
 %dir %{_datadir}/icons/hicolor
 %dir %{_datadir}/icons/hicolor/16x16
 %dir %{_datadir}/icons/hicolor/16x16/apps
@@ -295,6 +296,8 @@
 %{_sysconfdir}/xdg/autostart/git-annex.desktop
 %{_mandir}/man1/git-annex*.1%{?ext_man}
 %{_mandir}/man1/git-remote-tor-annex.1%{?ext_man}
+%{_mandir}/man1/git-remote-annex.1%{?ext_man}
+%{_mandir}/man1/git-remote-p2p-annex.1%{?ext_man}
 %{_datadir}/applications/git-annex.desktop
 %{_datadir}/icons/hicolor/16x16/apps/git-annex.png
 %{_datadir}/icons/hicolor/scalable/apps/git-annex.svg
