Index: git-annex.spec
===================================================================
--- git-annex.spec	(revision 35)
+++ git-annex.spec	(working copy)
@@ -23,6 +23,7 @@
 License:        AGPL-3.0-or-later AND GPL-3.0-or-later AND BSD-2-Clause AND MIT AND GPL-2.0-only
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://github.com/peti/git-annex/archive/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Patch0:         0001-enable-extensions-needed-by-new-version-of-persisten.patch
 BuildRequires:  bash-completion
 BuildRequires:  chrpath
 BuildRequires:  curl
@@ -168,7 +169,7 @@
 Optional dependency offering bash completion for git-annex
 
 %prep
-%autosetup
+%autosetup -p1
 
 %build
 %define cabal_configure_options -ftestsuite
