Index: git-annex.spec
===================================================================
--- git-annex.spec	(revision 23)
+++ git-annex.spec	(working copy)
@@ -138,6 +138,7 @@
 Recommends:     rsync
 Recommends:     ssh
 Suggests:       %{name}-bash-completion
+Patch01:        0001-fix-build-with-dlist-1.0.patch
 
 %description
 Git-annex allows managing files with git, without checking the file contents
@@ -169,6 +170,7 @@
 
 %prep
 %setup -q
+%patch01 -p1
 
 %build
 %define cabal_configure_options -ftestsuite
