Index: git-annex.spec
===================================================================
--- git-annex.spec	(revision 13)
+++ git-annex.spec	(working copy)
@@ -168,6 +168,8 @@
 %install
 %ghc_bin_install
 make DESTDIR=%{buildroot} BUILDER=./Setup install-misc install-mans
+rm %{buildroot}/usr/share/fish/completions/git-annex.fish
+rm %{buildroot}/usr/share/zsh/vendor-completions/_git-annex
 
 %check
 make %{?_smp_mflags} DESTDIR=%{buildroot} BUILDER=./Setup test

