Index: git-annex.spec
===================================================================
--- git-annex.spec	(revision 2)
+++ git-annex.spec	(working copy)
@@ -20,7 +20,7 @@
 Version:        6.20160511
 Release:        0
 Summary:        Manage files with git, without checking their contents into git
-License:        GPL-3.0+
+License:        GPL-3.0+ and AGPL-3.0+
 Group:          System/Libraries
 Url:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
@@ -115,6 +115,22 @@
 BuildRequires:  ghc-yesod-static-devel
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 # End cabal-rpm deps
+# Manually added dependencies which the Cabal file does not declare
+BuildRequires:  git-core
+BuildRequires:  rsync
+Requires:       git-core
+BuildRequires:  bash-completion
+BuildRequires:  curl
+Requires(post): desktop-file-utils
+Requires(post): hicolor-icon-theme
+Requires(postun): desktop-file-utils
+Requires(postun): hicolor-icon-theme
+Recommends:     curl
+Recommends:     gpg2
+Recommends:     lsof
+Recommends:     rsync
+Recommends:     ssh
+Suggests:       %{name}-bash-completion
 
 %description
 Git-annex allows managing files with git, without checking the file contents
@@ -134,6 +150,16 @@
 transferring them to other computers. The git-annex webapp makes it easy to set
 up and use git-annex this way.
 
+%package bash-completion
+Summary:        Bash completion for git-annex
+Group:          System/Shells
+Requires:       %{name} = %{version}
+Requires:       bash-completion
+Supplements:    packageand(%{name}:bash-completion)
+
+%description bash-completion
+Optional dependency offering bash completion for git-annex
+
 %prep
 %setup -q
 
@@ -141,15 +167,39 @@
 %build
 %ghc_bin_build
 
-
 %install
 %ghc_bin_install
-
+make DESTDIR=%{buildroot} BUILDER=./Setup install-misc install-mans
+
+%check
+make %{?_smp_mflags} DESTDIR=%{buildroot} BUILDER=./Setup test
+
+%post
+%desktop_database_post
+%icon_theme_cache_post
+
+%postun
+%desktop_database_postun
+%icon_theme_cache_postun
 
 %files
 %defattr(-,root,root,-)
-%doc COPYRIGHT
-%doc CHANGELOG NEWS README doc
+%doc COPYRIGHT README
 %{_bindir}/%{name}
+%{_bindir}/%{name}-shell
+%dir %{_datadir}/icons/hicolor
+%dir %{_datadir}/icons/hicolor/16x16
+%dir %{_datadir}/icons/hicolor/16x16/apps
+%dir %{_datadir}/icons/hicolor/scalable
+%dir %{_datadir}/icons/hicolor/scalable/apps
+%{_sysconfdir}/xdg/autostart/git-annex.desktop
+%{_mandir}/man1/git-annex*.1%{ext_man}
+%{_datadir}/applications/git-annex.desktop
+%{_datadir}/icons/hicolor/16x16/apps/git-annex.png
+%{_datadir}/icons/hicolor/scalable/apps/git-annex.svg
+
+%files bash-completion
+%defattr(-,root,root)
+%{_datadir}/bash-completion/completions/git-annex
 
 %changelog

