Index: git-annex.spec
===================================================================
--- git-annex.spec	(revision 10)
+++ git-annex.spec	(working copy)
@@ -113,6 +113,7 @@
 BuildRequires:  ghc-yesod-static-devel
 # Manually added dependencies which the Cabal file does not declare
 BuildRequires:  git-core
+BuildRequires:  chrpath
 BuildRequires:  rsync
 Requires:       git-core
 Requires(post): desktop-file-utils

