Index: git-annex.spec
===================================================================
--- git-annex.spec	(revision 23)
+++ git-annex.spec	(working copy)
@@ -24,6 +24,7 @@
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://github.com/opensuse-haskell/git-annex/archive/%{version}.tar.gz#/%{name}-%{version}.tar.gz
 Patch0:         0001-persistent-2.12.0.1-include-the-code-from-persistent.patch
+Patch1:         0001-Build-with-ram-instead-of-memory-package-to-support-.patch
 BuildRequires:  bash-completion
 BuildRequires:  chrpath
 BuildRequires:  curl
