Index: ghc-glaze.spec
===================================================================
--- ghc-glaze.spec	(revision 1)
+++ ghc-glaze.spec	(working copy)
@@ -31,7 +31,8 @@
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 
 %description
-Framework for rendering things with metadata/headers and values.
+A Haskell library framework for rendering things (or list of things, or things
+that contain things) that have metadata/header information as well as values.
 
 %package devel
 Summary:        Haskell %{pkg_name} library development files

