Index: ghc-glaze.spec
===================================================================
--- ghc-glaze.spec	(revision 1)
+++ ghc-glaze.spec	(working copy)
@@ -31,6 +31,7 @@
 
 %description
-Framework for rendering things with metadata/headers and values.
+A Haskell library framework for rendering things (or list of things, or things
+that contain things) that have metadata/header information as well as values.
 
 %package devel
 Summary:        Haskell %{pkg_name} library development files

