Index: ghc-servant-server.spec
===================================================================
--- ghc-servant-server.spec	(revision 5)
+++ ghc-servant-server.spec	(working copy)
@@ -94,6 +94,7 @@
 %autosetup -n %{pkg_name}-%{version}
 cabal-tweak-dep-ver base64-bytestring '< 1.2' '< 1.3'
 cabal-tweak-dep-ver http-api-data '< 0.4.3' '< 0.5'
+cabal-tweak-dep-ver base '< 4.15' '< 4.16'
 
 %build
 %ghc_lib_build
