Index: ghc-comonad.spec
===================================================================
--- ghc-comonad.spec	(revision 2)
+++ ghc-comonad.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        4.2.7.2
 Release:        0
 Summary:        Comonads
-License:        BSD-3-Clause
+License:        BSD-2-Clause
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

