Index: quickbench.spec
===================================================================
--- quickbench.spec	(revision 2)
+++ quickbench.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        1.0
 Release:        0
 Summary:        Quick & easy benchmarking of command-line programs
-License:        GPL-1.0-or-later
+License:        GPL-3.0-or-later
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
 BuildRequires:  chrpath
@@ -88,8 +88,6 @@
 
 %files -n ghc-%{name} -f ghc-%{name}.files
-%license LICENSE
 
 %files -n ghc-%{name}-devel -f ghc-%{name}-devel.files
-%doc CHANGES README.md
 
 %changelog

