Index: ghc-hamilton.spec
===================================================================
--- ghc-hamilton.spec	(revision 2)
+++ ghc-hamilton.spec	(working copy)
@@ -44,9 +44,7 @@
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 
 %description
-See README.md (or read online at <https://github.com/mstksg/hamilton#readme>).
-
-Documentation online at <https://mstksg.github.io/hamilton/>.
+Physics on generalized coordinate systems using Hamiltonian Mechanics and AD.
 
 %package devel
 Summary:        Haskell %{pkg_name} library development files

