Index: ghc-etcd.spec
===================================================================
--- ghc-etcd.spec	(revision 3)
+++ ghc-etcd.spec	(working copy)
@@ -22,6 +22,6 @@
 Version:        1.0.5
 Release:        0
 Summary:        Client for etcd, a highly-available key value store
-License:        Unknown
+License:        Unlicense
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

