Index: ghc-gogol-core.spec
===================================================================
--- ghc-gogol-core.spec	(revision 7)
+++ ghc-gogol-core.spec	(working copy)
@@ -22,6 +22,6 @@
 Version:        0.1.0
 Release:        0
 Summary:        Core data types and functionality for Gogol libraries
-License:        Unknown
+License:        MPL-2.0
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

