Index: ghc-hruby.spec
===================================================================
--- ghc-hruby.spec	(revision 1)
+++ ghc-hruby.spec	(working copy)
@@ -62,6 +62,7 @@
 %setup -q -n %{pkg_name}-%{version}
 
 %build
+%define cabal_configure_options --extra-include-dir=%{_includedir}/ruby-2.4.0 --extra-include-dir=%{_includedir}/ruby-2.4.0/%{_target}%{_gnu} --configure-option="--rubyversion=24 --rubylib=ruby2.4"
 %ghc_lib_build
 
 %install

