Index: xmobar.spec
===================================================================
--- xmobar.spec	(revision 1)
+++ xmobar.spec	(working copy)
@@ -34,6 +35,8 @@
 BuildRequires:  ghc-X11-xft-prof
 BuildRequires:  ghc-aeson-devel
 BuildRequires:  ghc-aeson-prof
+BuildRequires:  ghc-libmpd-devel
+BuildRequires:  ghc-libmpd-prof
 BuildRequires:  ghc-async-devel
 BuildRequires:  ghc-async-prof
 BuildRequires:  ghc-base-devel
