Index: xmobar.spec
===================================================================
--- xmobar.spec	(revision 5)
+++ xmobar.spec	(working copy)
@@ -91,6 +91,8 @@
 
 %prep
 %setup -q
+echo > Setup.hs 'import Distribution.Simple'
+echo >>Setup.hs 'main = defaultMain'
 
 %build
 %define cabal_configure_options -fwith_dbus -fwith_inotify -fwith_iwlib -fwith_mpris -fwith_thread -fwith_utf8 -fwith_xft -fwith_xpm

