Index: xmobar.spec
===================================================================
--- xmobar.spec	(revision 3)
+++ xmobar.spec	(working copy)
@@ -26,6 +26,7 @@
 License:        BSD-3-Clause
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
+Patch1:         0001-Fix-build-errors-with-base-4.18.x-and-mtl-2.3.x.patch
 BuildRequires:  chrpath
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-X11-devel
@@ -152,7 +153,7 @@
 This package provides the Haskell %{pkg_name} profiling library.
 
 %prep
-%autosetup
+%autosetup -p1
 
 %build
 %define cabal_configure_options -fall_extensions
