Index: ghc-pcre-light.spec
===================================================================
--- ghc-pcre-light.spec	(revision 2)
+++ ghc-pcre-light.spec	(working copy)
@@ -32,6 +32,9 @@
 # End cabal-rpm deps
 BuildRequires:  pkgconfig
 BuildRequires:  pkgconfig(libpcre)
+BuildRequires:  pkgconfig(libpcre16)
+BuildRequires:  pkgconfig(libpcrecpp)
+BuildRequires:  pkgconfig(libpcreposix)
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 
 %description
@@ -55,6 +58,9 @@
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
 # End cabal-rpm deps
+Requires:       pkgconfig(libpcre16)
+Requires:       pkgconfig(libpcrecpp)
+Requires:       pkgconfig(libpcreposix)
 
 %description devel
 This package provides the Haskell %{pkg_name} library development files.

