Index: ghc-amqp.spec
===================================================================
--- ghc-amqp.spec	(revision 4)
+++ ghc-amqp.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        0.14.1
 Release:        0
 Summary:        Client library for AMQP servers (currently only RabbitMQ)
-License:        BSD-3-Clause
+License:        MIT
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

