Index: ghc-iso3166-country-codes.spec
===================================================================
--- ghc-iso3166-country-codes.spec	(revision 3)
+++ ghc-iso3166-country-codes.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.20140203.8
 Release:        0
 Summary:        A datatype for ISO 3166 country codes
-License:        LGPL-2.0+
+License:        LGPL-3.0+
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

