Index: ghc-gogol-apps-tasks.spec
===================================================================
--- ghc-gogol-apps-tasks.spec	(revision 5)
+++ ghc-gogol-apps-tasks.spec	(working copy)
@@ -21,6 +21,6 @@
 Version:        0.1.0
 Release:        0
 Summary:        Google Tasks SDK
-License:        Unknown
+License:        MPL-2.0
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

