Index: shelltestrunner.spec
===================================================================
--- shelltestrunner.spec	(revision 1)
+++ shelltestrunner.spec	(working copy)
@@ -20,7 +20,7 @@
 Version:        1.9
 Release:        0
 Summary:        Easy, repeatable testing of CLI programs/commands
-License:        GPL-1.0-or-later
+License:        GPL-3.0-only
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz

