Index: xmonad.spec
===================================================================
--- xmonad.spec	(revision 26)
+++ xmonad.spec	(working copy)
@@ -25,6 +25,7 @@
 License:        BSD-3-Clause
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
+Source10:       xmonad.desktop
 BuildRequires:  chrpath
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-X11-devel
@@ -39,6 +40,11 @@
 BuildRequires:  ghc-time-devel
 BuildRequires:  ghc-transformers-devel
 BuildRequires:  ghc-unix-devel
+BuildRequires:  pkgconfig
+BuildRequires:  update-desktop-files
+BuildRequires:  pkgconfig(xinerama)
+Suggests:       ghc-xmonad-contrib-devel
+Suggests:       ghc-xmonad-devel
 %if %{with tests}
 BuildRequires:  ghc-QuickCheck-devel
 BuildRequires:  ghc-quickcheck-classes-devel
@@ -78,6 +84,11 @@
 
 %install
 %ghc_lib_install
+install -m0644 -D man/xmonad.1 %{buildroot}%{_mandir}/man1/xmonad.1
+gzip %{buildroot}%{_mandir}/man1/xmonad.1
+%define desktop_src %{buildroot}%{_datadir}/xsessions/xmonad.desktop
+install -m0644 -D %{SOURCE10} %{desktop_src}
+%suse_update_desktop_file %{desktop_src}
 %ghc_fix_rpath %{pkg_name}-%{version}
 
 %check
@@ -93,6 +104,8 @@
 %license LICENSE
 %doc CHANGES.md README.md
 %{_bindir}/%{name}
+%{_mandir}/man1/xmonad.1%{?ext_man}
+%{_datadir}/xsessions/xmonad.desktop
 
 %files -n ghc-%{name} -f ghc-%{name}.files
 %license LICENSE
