Index: xmonad.spec
===================================================================
--- xmonad.spec	(revision 7)
+++ xmonad.spec	(working copy)
@@ -26,7 +26,8 @@
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
 Source1:        https://hackage.haskell.org/package/%{name}-%{version}/revision/1.cabal#/%{name}.cabal
+Source10:       xmonad.desktop
 BuildRequires:  chrpath
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-X11-devel
@@ -40,6 +41,11 @@
 BuildRequires:  ghc-setlocale-devel
 BuildRequires:  ghc-unix-devel
 BuildRequires:  ghc-utf8-string-devel
+BuildRequires:  pkgconfig
+BuildRequires:  update-desktop-files
+BuildRequires:  pkgconfig(xinerama)
+Suggests:       ghc-xmonad-contrib-devel
+Suggests:       ghc-xmonad-devel
 %if %{with tests}
 BuildRequires:  ghc-QuickCheck-devel
 %endif
@@ -80,6 +86,15 @@
 
 %install
 %ghc_lib_install
+install -d %{buildroot}%{_mandir}/man1/
+mv %{buildroot}%{_datadir}/%{name}-%{version}/man/%{name}.1 %{buildroot}%{_mandir}/man1/
+gzip %{buildroot}%{_mandir}/man1/%{name}.1
+# Gzipping here because when we gzip it in %%build step, it breaks cabal setup
+
+%define desktop_src %{buildroot}%{_datadir}/xsessions/xmonad.desktop
+install -m0644 -D %{SOURCE10} %{desktop_src}
+%suse_update_desktop_file %{desktop_src}
+
 %ghc_fix_rpath %{pkg_name}-%{version}
 
 %check
@@ -97,8 +112,9 @@
 %{_bindir}/%{name}
 %dir %{_datadir}/%{name}-%{version}
 %dir %{_datadir}/%{name}-%{version}/man
-%{_datadir}/%{name}-%{version}/man/xmonad.1
+%{_mandir}/man1/xmonad.1%{ext_man}
 %{_datadir}/%{name}-%{version}/man/xmonad.1.html
+%{_datadir}/xsessions/xmonad.desktop
 %{_datadir}/%{name}-%{version}/man/xmonad.hs
 
 %files -n ghc-%{name} -f ghc-%{name}.files

