Index: xmonad.spec
===================================================================
--- xmonad.spec	(revision 3)
+++ xmonad.spec	(working copy)
@@ -27,6 +27,7 @@
 Url:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
 Source10:       xmonad.desktop
+Patch1:         https://github.com/xmonad/xmonad/pull/91.patch#/remove-state-file-after-startup.patch
 BuildRequires:  chrpath
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-X11-devel
@@ -84,6 +85,7 @@
 
 %prep
 %setup -q
+%patch1 -p1
 
 %build
 %ghc_lib_build

