Index: ghc-authenticate-oauth.spec
===================================================================
--- ghc-authenticate-oauth.spec	(revision 3)
+++ ghc-authenticate-oauth.spec	(working copy)
@@ -21,6 +21,6 @@
 Version:        1.5.1.2
 Release:        0
 Summary:        Library to authenticate with OAuth for Haskell web applications
-License:        BSD-3-Clause
+License:        BSD-2-Clause
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

