Index: alex.spec
===================================================================
--- alex.spec	(revision 3)
+++ alex.spec	(working copy)
@@ -16,12 +16,9 @@
 #
 
 
-# Disable tests on aarch64. See: https://github.com/simonmar/alex/issues/130
-%ifarch aarch64
+# Disable tests on ghc-8.8.x. See: https://github.com/simonmar/alex/issues/140.
 %bcond_with tests
-%else
-%bcond_without tests
-%endif
+
 Name:           alex
 Version:        3.2.4
 Release:        0

