Index: ghc-bloomfilter.spec
===================================================================
--- ghc-bloomfilter.spec	(revision 1)
+++ ghc-bloomfilter.spec	(working copy)
@@ -38,6 +38,7 @@
 BuildRequires:  ghc-test-framework-devel
 BuildRequires:  ghc-test-framework-quickcheck2-devel
 %endif
+Patch1:         0001-Adapt-FastShift-instances-for-Word32-to-ghc-9.4.x.patch
 
 %description
 Pure and impure Bloom Filter implementations.
@@ -53,8 +54,9 @@
 This package provides the Haskell %{pkg_name} library development files.
 
 %prep
-%autosetup -n %{pkg_name}-%{version}
+%autosetup -p1 -n %{pkg_name}-%{version}
 cp -p %{SOURCE1} %{pkg_name}.cabal
+cabal-tweak-dep-ver base '>= 4.4 && < 4.16' '< 5'
 
 %build
 %ghc_lib_build
