Index: ghc-fuzzcheck.spec
===================================================================
--- ghc-fuzzcheck.spec	(revision 1)
+++ ghc-fuzzcheck.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        0.1.1
 Release:        0
 Summary:        A simple checker for stress testing monadic code
-License:        BSD-3-Clause
+License:        MIT
 Group:          Development/Languages/Other
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

