Index: ghc-charsetdetect-ae.spec
===================================================================
--- ghc-charsetdetect-ae.spec	(revision 4)
+++ ghc-charsetdetect-ae.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        1.1.0.3
 Release:        0
 Summary:        Character set detection using Mozilla's Universal Character Set Detector
-License:        LGPL-2.0+
+License:        LGPL-2.1+
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

