Index: ghc-gnutls.spec
===================================================================
--- ghc-gnutls.spec	(revision 4)
+++ ghc-gnutls.spec	(working copy)
@@ -31,7 +31,6 @@
 BuildRequires:  ghc-monads-tf-devel
 BuildRequires:  ghc-rpm-macros
 BuildRequires:  ghc-transformers-devel
-BuildRequires:  libgnutls-devel
 # End cabal-rpm deps
 BuildRequires:  pkgconfig
 BuildRequires:  pkgconfig(gnutls)
@@ -50,7 +49,6 @@
 Requires:       %{name} = %{version}-%{release}
 Requires:       ghc-compiler = %{ghc_version}
 # Begin cabal-rpm deps:
-Requires:       libgnutls-devel
 Requires:       pkgconfig
 Requires:       pkgconfig(gnutls)
 Requires(post): ghc-compiler = %{ghc_version}

