Index: ghc-gnutls.spec
===================================================================
--- ghc-gnutls.spec	(revision 4)
+++ ghc-gnutls.spec	(working copy)
@@ -31,6 +31,5 @@
 BuildRequires:  ghc-monads-tf-devel
 BuildRequires:  ghc-rpm-macros
 BuildRequires:  ghc-transformers-devel
-BuildRequires:  libgnutls-devel
 BuildRequires:  pkgconfig
 BuildRequires:  pkgconfig(gnutls)
@@ -50,6 +49,5 @@
 Requires:       %{name} = %{version}-%{release}
 Requires:       ghc-compiler = %{ghc_version}
-Requires:       libgnutls-devel
 Requires:       pkgconfig
 Requires:       pkgconfig(gnutls)
 Requires(post): ghc-compiler = %{ghc_version}

