Index: ghc-ihaskell-inline-r.spec
===================================================================
--- ghc-ihaskell-inline-r.spec	(revision 5)
+++ ghc-ihaskell-inline-r.spec	(working copy)
@@ -37,5 +37,7 @@
 BuildRequires:  ghc-template-haskell-devel
 BuildRequires:  ghc-temporary-devel
+# Does not build in 32 bit platforms: https://github.com/tweag/HaskellR/issues/258.
+ExcludeArch:    %{ix86}
 
 %description
 Embed R quasiquotes and plots in IHaskell notebooks.

