Index: ghc-time-compat.spec
===================================================================
--- ghc-time-compat.spec	(revision 2)
+++ ghc-time-compat.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.1.0.3
 Release:        0
 Summary:        Compatibility with old-time for the time package
-License:        BSD-3-Clause
+License:        BSD-2-Clause
 Group:          Development/Languages/Haskell
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

