Index: gtk2hs-buildtools.spec
===================================================================
--- gtk2hs-buildtools.spec	(revision 1)
+++ gtk2hs-buildtools.spec	(working copy)
@@ -38,6 +38,7 @@
 BuildRequires:  ghc-rpm-macros
 BuildRequires:  happy
 ExcludeArch:    %{ix86}
+Patch1:         fix-ghc-9.4.x-build.patch
 
 %description
 This package provides a set of helper programs necessary to build the Gtk2Hs
