Index: ghc-turtle-options.spec
===================================================================
--- ghc-turtle-options.spec	(revision 3)
+++ ghc-turtle-options.spec	(working copy)
@@ -64,7 +64,7 @@
 
 %install
 %ghc_lib_install
-%ghc_fix_rpath %{pkg_name}-%{version}
+rm %{buildroot}%{_bindir}/example
 
 
 %check
@@ -83,5 +82,4 @@
 %files -f %{name}.files
 %doc LICENSE
-%{_bindir}/example
 
 %files devel -f %{name}-devel.files

