Index: ghc-turtle-options.spec
===================================================================
--- ghc-turtle-options.spec	(revision 3)
+++ ghc-turtle-options.spec	(working copy)
@@ -64,7 +64,7 @@
 
 %install
 %ghc_lib_install
-%{ghc_fix_rpath} %{pkg_name}-%{version}
+rm %{buildroot}%{_bindir}/example
 
 
 %check
@@ -83,7 +82,6 @@
 %files -f %{name}.files
 %defattr(-,root,root,-)
 %doc LICENSE
-%{_bindir}/example
 
 %files devel -f %{name}-devel.files
 %defattr(-,root,root,-)

