Index: ghc-hackage-security.spec
===================================================================
--- ghc-hackage-security.spec	(revision 1)
+++ ghc-hackage-security.spec	(working copy)
@@ -89,6 +89,8 @@
 %prep
 %autosetup -n %{pkg_name}-%{version}
 cp -p %{SOURCE1} %{pkg_name}.cabal
+cabal-tweak-dep-ver base '< 4.15' '<5'
+cabal-tweak-dep-ver template-haskell '< 2.17' '<3'
 
 %build
 %ghc_lib_build
