Index: ghc-cmark.spec
===================================================================
--- ghc-cmark.spec	(revision 3)
+++ ghc-cmark.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        0.6
 Release:        0
 Summary:        Fast, accurate CommonMark (Markdown) parser and renderer
-License:        BSD-3-Clause
+License:        BSD-3-Clause AND BSD-2-Clause AND MIT
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
 BuildRequires:  ghc-Cabal-devel
