From d332734962298fe44dfac488af374399046600a2 Mon Sep 17 00:00:00 2001
From: Peter Simons <simons@cryp.to>
Date: Thu, 21 Jul 2016 11:22:20 +0200
Subject: [PATCH 2/2] Fix build on older SUSE distributions like Leap or SLE12

---
 ghc-cmark.spec | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/ghc-cmark.spec b/ghc-cmark.spec
index b1d2e14..46b7b4b 100644
--- a/ghc-cmark.spec
+++ b/ghc-cmark.spec
@@ -17,6 +17,13 @@
 
 
 %global pkg_name cmark
+%if 0%{?suse_version} >= 1330
+%define is_tumble 1
+%endif
+# is Leap 42.2+ or SLE12SP2+
+%if 0%{?sle_version} >= 120200
+%define is_sp2up 1
+%endif
 %bcond_with tests
 Name:           ghc-%{pkg_name}
 Version:        0.5.2.1
@@ -31,6 +38,10 @@ BuildRequires:  ghc-bytestring-devel
 BuildRequires:  ghc-rpm-macros
 BuildRequires:  ghc-text-devel
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+%if 0%{?is_tumble} || 0%{?is_sp2up}
+BuildRequires:  pkgconfig
+BuildRequires:  pkgconfig(libcmark)
+%endif
 %if %{with tests}
 BuildRequires:  ghc-HUnit-devel
 %endif
@@ -48,12 +59,19 @@ Requires:       %{name} = %{version}-%{release}
 Requires:       ghc-compiler = %{ghc_version}
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
+%if 0%{?is_tumble} || 0%{?is_sp2up}
+Requires:       pkgconfig
+Requires:       pkgconfig(libcmark)
+%endif
 
 %description devel
 This package provides the Haskell %{pkg_name} library development files.
 
 %prep
 %setup -q -n %{pkg_name}-%{version}
+%if 0%{?is_tumble} || 0%{?is_sp2up}
+cabal-tweak-flag    pkgconfig   True
+%endif
 
 
 %build
-- 
2.9.0

