Index: ghc-cmark.spec
===================================================================
--- ghc-cmark.spec	(revision 2)
+++ ghc-cmark.spec	(working copy)
@@ -17,12 +17,19 @@
 
 
 %global pkg_name cmark
+%if 0%{?suse_version} >= 1330
+%define is_tumble 1
+%endif
+# is Leap 42.2+ or SLE12SP2+
+%if 0%{?sle_version} >= 120200
+%define is_sp2up 1
+%endif
 %bcond_with tests
 Name:           ghc-%{pkg_name}
 Version:        0.5.2.1
 Release:        0
 Summary:        Fast, accurate CommonMark (Markdown) parser and renderer
-License:        BSD-3-Clause
+License:        BSD-3-Clause and BSD-2-Clause and MIT
 Group:          System/Libraries
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
@@ -32,6 +39,10 @@
 BuildRequires:  ghc-rpm-macros
 BuildRequires:  ghc-text-devel
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+%if 0%{?is_tumble} || 0%{?is_sp2up}
+BuildRequires:  pkgconfig
+BuildRequires:  pkgconfig(libcmark)
+%endif
 %if %{with tests}
 BuildRequires:  ghc-HUnit-devel
 %endif
@@ -48,15 +59,21 @@
 Group:          Development/Libraries/Other
 Requires:       %{name} = %{version}-%{release}
 Requires:       ghc-compiler = %{ghc_version}
+Requires:       pkgconfig
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
+%if 0%{?is_tumble} || 0%{?is_sp2up}
+Requires:       pkgconfig(libcmark)
+%endif
 
 %description devel
 This package provides the Haskell %{pkg_name} library development files.
 
 %prep
 %setup -q -n %{pkg_name}-%{version}
-
+%if 0%{?is_tumble} || 0%{?is_sp2up}
+cabal-tweak-flag    pkgconfig   True
+%endif
 
 %build
 %ghc_lib_build

