Index: ghc-applicative-quoters.spec
===================================================================
--- ghc-applicative-quoters.spec (revision 1)
+++ ghc-applicative-quoters.spec (revision 2)
@@ -31,6 +31,8 @@
 BuildRequires:  ghc-rpm-macros
 BuildRequires:  ghc-template-haskell-devel
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+# https://patch-diff.githubusercontent.com/raw/bmillwood/applicative-quoters/pull/7.patch
+Patch1:         ghc-8-support.patch
 
 %description
 Quasiquoters taken from Matt Morrow's haskell-src-meta to implement Conor
@@ -55,6 +57,8 @@
 %prep
 %setup -q -n %{pkg_name}-%{version}
 cp -p %{SOURCE1} %{pkg_name}.cabal
+%patch1 -p1
+cabal-tweak-dep-ver template-haskell '< 2.11' ' < 2.12'
 
 %build
 %ghc_lib_build

