Index: ghc-amazonka-s3-streaming.spec
===================================================================
--- ghc-amazonka-s3-streaming.spec	(revision 2)
+++ ghc-amazonka-s3-streaming.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.1.0.4
 Release:        0
 Summary:        Provides conduits to upload data to S3 using the Multipart API
-License:        BSD-3-Clause
+License:        MIT
 Group:          Development/Languages/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

