Index: ghc-gogol-play-moviespartner.spec
===================================================================
--- ghc-gogol-play-moviespartner.spec	(revision 5)
+++ ghc-gogol-play-moviespartner.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.1.0
 Release:        0
 Summary:        Google Play Movies Partner SDK
-License:        Unknown
+License:        MPL-2.0
 Group:          Development/Languages/Haskell
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

