Index: ghc-mountpoints.spec
===================================================================
--- ghc-mountpoints.spec	(revision 3)
+++ ghc-mountpoints.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        1.0.2
 Release:        0
 Summary:        List mount points
-License:        LGPL-2.0+
+License:        LGPL-2.1+ and BSD-3-Clause
 Group:          System/Libraries
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
@@ -65,7 +65,7 @@
 
 %files -f %{name}.files
 %defattr(-,root,root,-)
-%doc LICENSE
+%doc LICENSE LGPL-2.1
 
 %files devel -f %{name}-devel.files
 %defattr(-,root,root,-)

