Index: ghc-csv-conduit.spec
===================================================================
--- ghc-csv-conduit.spec	(revision 10)
+++ ghc-csv-conduit.spec	(working copy)
@@ -26,7 +26,6 @@
 Group:          Development/Languages/Other
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
-BuildRequires:  chrpath
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-array-devel
 BuildRequires:  ghc-attoparsec-devel
@@ -100,10 +99,9 @@
 
 %build
-%define cabal_configure_options -fbench
+%define cabal_configure_options -f-bench
 %ghc_lib_build
 
 %install
 %ghc_lib_install
-%{ghc_fix_rpath} %{pkg_name}-%{version}
 
 %check
@@ -122,7 +120,6 @@
 %files -f %{name}.files
 %defattr(-,root,root,-)
 %doc LICENSE
-%{_bindir}/bench
 
 %files devel -f %{name}-devel.files
 %defattr(-,root,root,-)

