Index: ghc-X11.spec
===================================================================
--- ghc-X11.spec	(revision 20)
+++ ghc-X11.spec	(working copy)
@@ -27,9 +27,13 @@
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-data-default-class-devel
 BuildRequires:  ghc-rpm-macros
-BuildRequires:  libX11-devel
-BuildRequires:  libXext-devel
-BuildRequires:  libXrandr-devel
+BuildRequires:  pkgconfig
+BuildRequires:  pkgconfig(x11)
+BuildRequires:  pkgconfig(x11-xcb)
+BuildRequires:  pkgconfig(xext)
+BuildRequires:  pkgconfig(xinerama)
+BuildRequires:  pkgconfig(xrandr)
+BuildRequires:  pkgconfig(xscrnsaver)
 
 %description
 A Haskell binding to the X11 graphics library. The binding is a direct
@@ -40,9 +44,12 @@
 Summary:        Haskell %{pkg_name} library development files
 Requires:       %{name} = %{version}-%{release}
 Requires:       ghc-compiler = %{ghc_version}
-Requires:       libX11-devel
-Requires:       libXext-devel
-Requires:       libXrandr-devel
+Requires:       pkgconfig(x11)
+Requires:       pkgconfig(x11-xcb)
+Requires:       pkgconfig(xext)
+Requires:       pkgconfig(xinerama)
+Requires:       pkgconfig(xrandr)
+Requires:       pkgconfig(xscrnsaver)
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
 
