Index: ghc-X11.spec
===================================================================
--- ghc-X11.spec	(revision 5)
+++ ghc-X11.spec	(working copy)
@@ -37,6 +37,7 @@
 BuildRequires:  pkgconfig(xext)
 BuildRequires:  pkgconfig(xinerama)
 BuildRequires:  pkgconfig(xrandr)
+BuildRequires:  pkgconfig(xscrnsaver)
 
 %description
 A Haskell binding to the X11 graphics library. The binding is a direct

