Index: ghc-X11.spec
===================================================================
--- ghc-X11.spec	(revision 2)
+++ ghc-X11.spec	(working copy)
@@ -34,5 +34,11 @@
 BuildRequires:  libXrandr-devel
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+BuildRequires:  pkgconfig
+BuildRequires:  pkgconfig(x11)
+BuildRequires:  pkgconfig(x11-xcb)
+BuildRequires:  pkgconfig(xext)
+BuildRequires:  pkgconfig(xinerama)
+BuildRequires:  pkgconfig(xrandr)
 
 %description
 A Haskell binding to the X11 graphics library. The binding is a direct

