Index: ghc-http-api-data.spec
===================================================================
--- ghc-http-api-data.spec	(revision 2)
+++ ghc-http-api-data.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        0.2.3
 Release:        0
 Summary:        Converting to/from HTTP API data like URL pieces, headers and query parameters
-License:        BSD-3-Clause
+License:        BSD-2-Clause
 Group:          Development/Languages/Haskell
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

