Index: ghc-avers.spec
===================================================================
--- ghc-avers.spec	(revision 2)
+++ ghc-avers.spec	(working copy)
@@ -60,7 +60,15 @@
 %endif
 
 %description
-Empty.
+This is the server-side implementation of the Avers storage model. It is a
+Haskell library, intended to be used by your application.
+
+The library provides you with everything you need to write your own Avers
+server. You can create and patch objects and collections thereof. It has
+builtin support for sessions, authentication and managing blobs.
+
+The database is not configurable. Avers is currently hardcoded to use
+RethinkDB.
 
 %package devel
 Summary:        Haskell %{pkg_name} library development files

