Index: xmonad.spec
===================================================================
--- xmonad.spec	(revision 7)
+++ xmonad.spec	(working copy)
@@ -26,6 +26,7 @@
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
 Source10:       xmonad.desktop
+Patch0:         xmonad-ghc9.patch
 BuildRequires:  chrpath
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-X11-devel
@@ -80,7 +81,7 @@
 This package provides the Haskell %{name} library development files.
 
 %prep
-%autosetup
+%autosetup -p1
 
 %build
 %ghc_lib_build
