Index: cpphs.spec
===================================================================
--- cpphs.spec	(revision 2)
+++ cpphs.spec	(working copy)
@@ -17,11 +17,11 @@
 
 
 %global pkg_name cpphs
-Name:           %{pkg_name}
+Name:           cpphs
 Version:        1.20.1
 Release:        0
 Summary:        A liberalised re-implementation of cpp, the C pre-processor
-License:        LGPL-2.0+
+License:        GPL-2.0+
 Group:          Development/Languages/Other
 Url:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
@@ -47,6 +47,7 @@
 
 %package -n ghc-%{name}
 Summary:        Haskell %{name} library
+License:        LGPL-2.1
 Group:          System/Libraries
 
 %description -n ghc-%{name}
@@ -54,6 +55,7 @@
 
 %package -n ghc-%{name}-devel
 Summary:        Haskell %{name} library development files
+License:        LGPL-2.1
 Group:          Development/Libraries/Other
 Requires:       ghc-%{name} = %{version}-%{release}
 Requires:       ghc-compiler = %{ghc_version}
@@ -66,14 +68,13 @@
 %prep
 %setup -q
 
-
 %build
 %ghc_lib_build
 
-
 %install
 %ghc_lib_install
-
+install -D --mode=444 docs/cpphs.1 %{buildroot}%{_mandir}/man1/cpphs.1
+chmod a-x LICENCE-* README docs/*
 
 %post -n ghc-%{name}-devel
 %ghc_pkg_recache
@@ -84,8 +85,9 @@
 %files
 %defattr(-,root,root,-)
 %doc LICENCE-LGPL
-%doc CHANGELOG LICENCE-GPL LICENCE-commercial README docs
+%doc LICENCE-GPL LICENCE-commercial README docs
 %{_bindir}/%{name}
+%{_mandir}/man1/cpphs.1%{ext_man}
 
 %files -n ghc-%{name} -f ghc-%{name}.files
 %defattr(-,root,root,-)
@@ -93,6 +95,6 @@
 
 %files -n ghc-%{name}-devel -f ghc-%{name}-devel.files
 %defattr(-,root,root,-)
-%doc CHANGELOG LICENCE-GPL LICENCE-commercial README docs
+%doc LICENCE-GPL LICENCE-commercial README docs
 
 %changelog

