Index: ghc-glib.spec
===================================================================
--- ghc-glib.spec	(revision 1)
+++ ghc-glib.spec	(working copy)
@@ -29,6 +29,7 @@
 # Begin cabal-rpm deps:
 BuildRequires:  ghc-bytestring-devel
 BuildRequires:  ghc-containers-devel
+BuildRequires:  ghc-gtk2hs-buildtools-devel
 BuildRequires:  ghc-rpm-macros
 BuildRequires:  ghc-text-devel
 BuildRequires:  ghc-utf8-string-devel

