Index: ghc-elm-export.spec
===================================================================
--- ghc-elm-export.spec	(revision 1)
+++ ghc-elm-export.spec	(working copy)
@@ -22,6 +22,6 @@
 Version:        0.6.0.0
 Release:        0
 Summary:        A library to generate Elm types from Haskell source
-License:        Unknown
+License:        EPL-1.0
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

