Index: ShellCheck.spec
===================================================================
--- ShellCheck.spec (revision 4)
+++ ShellCheck.spec (revision 3)
@@ -25,7 +25,7 @@
 License:        GPL-3.0-or-later
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{name}
-Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
 BuildRequires:  chrpath
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-QuickCheck-devel

