Index: ghc-polyparse.spec
===================================================================
--- ghc-polyparse.spec	(revision 4)
+++ ghc-polyparse.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        1.12
 Release:        0
 Summary:        A variety of alternative parser combinator libraries
-License:        LGPL-2.0+
+License:        LGPL-2.1
 Group:          Development/Languages/Other
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

