Index: ghc-network-transport-zeromq.spec
===================================================================
--- ghc-network-transport-zeromq.spec	(revision 1)
+++ ghc-network-transport-zeromq.spec	(working copy)
@@ -49,5 +49,7 @@
 BuildRequires:  ghc-tasty-hunit-devel
 %endif
+BuildRequires:  ghc-criterion-devel
+BuildRequires:  ghc-distributed-process-devel
 
 %description
 Implementation of the <http://hackage.haskell.org/package/network-transport

