Index: ghc-crypto-numbers.spec
===================================================================
--- ghc-crypto-numbers.spec	(revision 3)
+++ ghc-crypto-numbers.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        0.2.7
 Release:        0
 Summary:        Cryptographic numbers: functions and algorithms
-License:        BSD-3-Clause
+License:        BSD-2-Clause
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

