Index: ghc-blosum.spec
===================================================================
--- ghc-blosum.spec	(revision 2)
+++ ghc-blosum.spec	(working copy)
@@ -21,6 +21,6 @@
 Version:        0.1.1.4
 Release:        0
 Summary:        BLOSUM generator
-License:        GPL-2.0-or-later
+License:        GPL-3.0-or-later
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

