Index: ghc-wai-predicates.spec
===================================================================
--- ghc-wai-predicates.spec	(revision 1)
+++ ghc-wai-predicates.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        0.9.0
 Release:        0
 Summary:        WAI request predicates
-License:        Unknown
+License:        MIT
 Group:          Development/Languages/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

