Index: ghc-vector-algorithms.spec
===================================================================
--- ghc-vector-algorithms.spec	(revision 6)
+++ ghc-vector-algorithms.spec	(working copy)
@@ -26,7 +26,6 @@
 Group:          Development/Languages/Other
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
-BuildRequires:  chrpath
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-bytestring-devel
 BuildRequires:  ghc-mtl-devel
@@ -60,10 +59,9 @@
 
 %build
-%define cabal_configure_options -fbench
+%define cabal_configure_options -f-bench
 %ghc_lib_build
 
 %install
 %ghc_lib_install
-%{ghc_fix_rpath} %{pkg_name}-%{version}
 
 %check
@@ -82,7 +80,6 @@
 %files -f %{name}.files
 %defattr(-,root,root,-)
 %doc LICENSE
-%{_bindir}/vector-algorithms-bench
 
 %files devel -f %{name}-devel.files
 %defattr(-,root,root,-)

