Index: ghc-leveldb-haskell.spec
===================================================================
--- ghc-leveldb-haskell.spec	(revision 1)
+++ ghc-leveldb-haskell.spec	(working copy)
@@ -45,6 +45,7 @@
 BuildRequires:  ghc-temporary-devel
 %endif
 # End cabal-rpm deps
+BuildRequires:   leveldb-devel
 
 %description
 From <http://code.google.com/p/snappy>:
@@ -66,6 +67,7 @@
 Requires:       ghc-compiler = %{ghc_version}
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
+Requires:       leveldb
 
 %description devel
 This package provides the Haskell %{pkg_name} library development

