Index: ghc-OpenGLRaw.spec
===================================================================
--- ghc-OpenGLRaw.spec	(revision 2)
+++ ghc-OpenGLRaw.spec	(working copy)
@@ -2,6 +2,7 @@
 # spec file for package ghc-OpenGLRaw
 #
 # Copyright (c) 2016 SUSE LINUX GmbH, Nuernberg, Germany.
+# Copyright (c) 2013 Peter Trommler peter.trommler@ohm-hochschule.de
 #
 # All modifications and additions to the file contributed by third parties
 # remain the property of their copyright owners, unless otherwise agreed
@@ -17,17 +18,19 @@
 
 
 %global pkg_name OpenGLRaw
-Name:           ghc-%{pkg_name}
+Name:           ghc-OpenGLRaw
 Version:        3.2.0.0
 Release:        0
 Summary:        A raw binding for the OpenGL graphics system
 License:        BSD-3-Clause
 Group:          System/Libraries
-Url:            https://hackage.haskell.org/package/%{pkg_name}
-Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
+Url:            http://hackage.haskell.org/package/%{pkg_name}
+Source0:        http://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
+Source1:        ghc-OpenGLRaw-rpmlintrc
 # Begin cabal-rpm deps:
 BuildRequires:  Mesa-libGL-devel
 BuildRequires:  ghc-Cabal-devel
+BuildRequires:  ghc-base-devel
 BuildRequires:  ghc-bytestring-devel
 BuildRequires:  ghc-containers-devel
 BuildRequires:  ghc-fixed-devel
@@ -36,7 +39,6 @@
 BuildRequires:  ghc-text-devel
 BuildRequires:  ghc-transformers-devel
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
-# End cabal-rpm deps
 
 %description
 OpenGLRaw is a raw Haskell binding for the OpenGL 4.5 graphics system and lots
@@ -59,11 +61,15 @@
 Summary:        Haskell %{pkg_name} library development files
 Group:          Development/Libraries/Other
 Requires:       %{name} = %{version}-%{release}
-# Begin cabal-rpm deps:
-Requires:       Mesa-libGL-devel
 Requires:       ghc-compiler = %{ghc_version}
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
+# Begin cabal-rpm deps:
+%if 0%{?suse_version} < 1230
+Requires:       Mesa-devel
+%else
+Requires:       Mesa-libGL-devel
+%endif
 # End cabal-rpm deps
 
 %description devel
@@ -72,14 +78,11 @@
 %prep
 %setup -q -n %{pkg_name}-%{version}
 
-
 %build
 %ghc_lib_build
 
-
 %install
 %ghc_lib_install
-
 
 %post devel
 %ghc_pkg_recache
@@ -93,6 +96,6 @@
 
 %files devel -f %{name}-devel.files
 %defattr(-,root,root,-)
-%doc CHANGELOG.md README.md
+%doc README.md
 
 %changelog
Index: _constraints
===================================================================
--- _constraints	(revision 0)
+++ _constraints	(revision 0)
@@ -0,0 +1,7 @@
+<constraints>
+  <hardware>
+    <physicalmemory>
+      <size unit="G">4</size>
+    </physicalmemory>
+  </hardware>
+</constraints>
Index: ghc-OpenGLRaw-rpmlintrc
===================================================================
--- ghc-OpenGLRaw-rpmlintrc	(revision 0)
+++ ghc-OpenGLRaw-rpmlintrc	(revision 0)
@@ -0,0 +1,1 @@
+addFilter("static-library-without-debuginfo")

