Index: cabal-install.spec
===================================================================
--- cabal-install.spec	(revision 11)
+++ cabal-install.spec	(working copy)
@@ -17,12 +17,13 @@
 
 
 Name:           cabal-install
-Version:        3.2.0.0
+Version:        3.4.0.0_rc4
 Release:        0
 Summary:        The command-line interface for Cabal and Hackage
 License:        BSD-3-Clause
 URL:            https://hackage.haskell.org/package/%{name}
-Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
+Source0:        https://hackage.haskell.org/package/%{name}-3.2.0.0/%{name}-3.2.0.0.tar.gz
+Patch1:         cabal-install-3.4.0.0-rc4.patch
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-HTTP-devel
 BuildRequires:  ghc-array-devel
@@ -56,6 +57,8 @@
 BuildRequires:  ghc-transformers-devel
 BuildRequires:  ghc-unix-devel
 BuildRequires:  ghc-zlib-devel
+BuildRequires:  ghc-regex-base-devel
+BuildRequires:  ghc-regex-posix-devel
 
 %description
 The 'cabal' command-line program simplifies the process of managing Haskell
@@ -63,7 +66,8 @@
 installation of Haskell libraries and programs.
 
 %prep
-%autosetup
+%autosetup -p1 -n cabal-install-3.2.0.0
+cabal-tweak-dep-ver base '< 4.15' '<5'
 
 %build
 %ghc_bin_build
