Index: ghc-half.spec
===================================================================
--- ghc-half.spec	(revision 2)
+++ ghc-half.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.2.2.3
 Release:        0
 Summary:        Half-precision floating-point
-License:        BSD-3-Clause
+License:        BSD-2-Clause
 Group:          Development/Languages/Haskell
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

