Index: ghc-hxt-expat.spec
===================================================================
--- ghc-hxt-expat.spec	(revision 5)
+++ ghc-hxt-expat.spec	(working copy)
@@ -21,6 +21,6 @@
 Version:        9.1.1
 Release:        0
 Summary:        Expat parser for HXT
-License:        Unknown
+License:        MIT
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

