Index: ghc-servant-js.spec
===================================================================
--- ghc-servant-js.spec	(revision 1)
+++ ghc-servant-js.spec	(working copy)
@@ -42,6 +42,8 @@
 BuildRequires:  ghc-servant-devel
 %endif
 # End cabal-rpm deps
+BuildRequires:  ghc-servant-server-devel
+BuildRequires:  ghc-warp-devel
 
 %description
 Automatically derive javascript functions to query servant webservices.

