Index: ghc-re2.spec
===================================================================
--- ghc-re2.spec	(revision 2)
+++ ghc-re2.spec	(working copy)
@@ -57,6 +57,7 @@
 Requires:       re2-devel
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
+BuildRequires:  gcc-c++
 
 %description devel
 This package provides the Haskell %{pkg_name} library development files.

