Index: hdevtools.spec
===================================================================
--- hdevtools.spec	(revision 2)
+++ hdevtools.spec	(working copy)
@@ -26,6 +26,5 @@
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
 BuildRequires:  ghc-Cabal-devel
-BuildRequires:  ghc-bin-package-db-devel
 BuildRequires:  ghc-cmdargs-devel
 BuildRequires:  ghc-directory-devel
 BuildRequires:  ghc-filepath-devel

