Index: ghc-rethinkdb.spec
===================================================================
--- ghc-rethinkdb.spec	(revision 7)
+++ ghc-rethinkdb.spec	(working copy)
@@ -67,7 +67,7 @@
 
 
 %build
-%define cabal_configure_options -fdev
+%define cabal_configure_options -f-dev
 %ghc_lib_build
 
 
@@ -88,7 +88,6 @@
 %files -f %{name}.files
 %defattr(-,root,root,-)
 %doc LICENSE
-%{_bindir}/proto2hs
 
 %files devel -f %{name}-devel.files
 %defattr(-,root,root,-)

