Index: ghc-language-dockerfile.spec
===================================================================
--- ghc-language-dockerfile.spec	(revision 1)
+++ ghc-language-dockerfile.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        0.3.5.0
 Release:        0
 Summary:        Dockerfile linter, parser, pretty-printer and embedded DSL
-License:        GPL-3.0-or-later
+License:        GPL-3.0
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

