Index: ghc-foundation.spec
===================================================================
--- ghc-foundation.spec	(revision 2)
+++ ghc-foundation.spec	(working copy)
@@ -26,6 +26,7 @@
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
 Source1:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/revision/1.cabal#/%{pkg_name}.cabal
+Patch1:         https://gitlab.haskell.org/ghc/head.hackage/-/raw/master/patches/foundation-0.0.25.patch
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-basement-devel
 BuildRequires:  ghc-rpm-macros
@@ -62,7 +63,7 @@
 This package provides the Haskell %{pkg_name} library development files.
 
 %prep
-%autosetup -n %{pkg_name}-%{version}
+%autosetup -n %{pkg_name}-%{version} -p1
 cp -p %{SOURCE1} %{pkg_name}.cabal
 
 %build
