diff --git a/pandoc.spec b/pandoc.spec
index 3df2553..86f2ced 100644
--- a/pandoc.spec
+++ b/pandoc.spec
@@ -28,6 +28,7 @@ Url:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
 Source10:       pandoc-pdf.readme
 BuildRequires:  chrpath
+BuildRequires:  fdupes
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-HTTP-devel
 BuildRequires:  ghc-JuicyPixels-devel
@@ -148,6 +149,8 @@ cp %{SOURCE10} .
 %install
 %ghc_lib_install
 %ghc_fix_rpath %{pkg_name}-%{version}
+# Link duplicate template files
+%fdupes %{buildroot}%{_datadir}/%{pkg_name}-%{version}/data/templates/
 
 %check
 %cabal_test
