Index: ghc-cheapskate.spec
===================================================================
--- ghc-cheapskate.spec	(revision 11)
+++ ghc-cheapskate.spec	(working copy)
@@ -71,10 +71,10 @@
 
 %build
-%define cabal_configure_options -fdingus
+%define cabal_configure_options -f-dingus
 %ghc_lib_build
 
 %install
 %ghc_lib_install
-%{ghc_fix_rpath} %{pkg_name}-%{version}
+%{ghc_fix_rpath} %{pkg_name}-%{version}
 
 %post devel
@@ -90,7 +90,6 @@
 %defattr(-,root,root,-)
 %doc LICENSE
 %{_bindir}/%{pkg_name}
-%{_bindir}/cheapskate-dingus
 
 %files devel -f %{name}-devel.files
 %defattr(-,root,root,-)

