Index: ghc-cabal-helper.spec
===================================================================
--- ghc-cabal-helper.spec	(revision 11)
+++ ghc-cabal-helper.spec	(working copy)
@@ -83,6 +83,7 @@
 %setup -q -n %{pkg_name}-%{version}
 
 %build
+%define cabal_configure_options --libexecdir=%{_libexecdir}
 %ghc_lib_build
 
 %install
@@ -100,7 +101,7 @@
 %files -f %{name}.files
 %defattr(-,root,root,-)
 %doc LICENSE
-%{_bindir}/cabal-helper-wrapper
+%{_libexecdir}/cabal-helper-wrapper
 
 %files devel -f %{name}-devel.files
 %defattr(-,root,root,-)

