Index: ghc-amazonka-shield.spec
===================================================================
--- ghc-amazonka-shield.spec	(revision 3)
+++ ghc-amazonka-shield.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        1.4.5
 Release:        0
 Summary:        Amazon Shield SDK
-License:        Unknown
+License:        MPL-2.0
 Group:          Development/Languages/Haskell
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

