Index: ghc-webpage.spec
===================================================================
--- ghc-webpage.spec	(revision 2)
+++ ghc-webpage.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.0.5
 Release:        0
 Summary:        Organized and simple web page scaffold for blaze and lucid
-License:        MIT
+License:        BSD-3-Clause
 Group:          Development/Languages/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

