Index: ghc-biofasta.spec
===================================================================
--- ghc-biofasta.spec	(revision 2)
+++ ghc-biofasta.spec	(working copy)
@@ -21,6 +21,6 @@
 Version:        0.0.3
 Release:        0
 Summary:        Library for reading fasta sequence files
-License:        GPL-1.0-or-later
+License:        GPL-3.0-only
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

