Index: ghc-imm.spec
===================================================================
--- ghc-imm.spec	(revision 10)
+++ ghc-imm.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        1.0.1.0
 Release:        0
 Summary:        Execute arbitrary actions for each unread element of RSS/Atom feeds
-License:        SUSE-Public-Domain
+License:        WTFPL
 Group:          Development/Languages/Haskell
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

