Index: ghc-wai-handler-launch.spec
===================================================================
--- ghc-wai-handler-launch.spec	(revision 1)
+++ ghc-wai-handler-launch.spec	(working copy)
@@ -39,7 +39,6 @@
 
 %description
-API docs and the README are available at
-<http://www.stackage.org/package/wai-handler-launch>.
+Launch a web app in the default browser.
 
 %package devel
 Summary:        Haskell %{pkg_name} library development files

