Index: ghc-gtk.spec
===================================================================
--- ghc-gtk.spec	(revision 3)
+++ ghc-gtk.spec	(working copy)
@@ -43,5 +43,6 @@
 BuildRequires:  pkgconfig(gthread-2.0)
 BuildRequires:  pkgconfig(gtk+-2.0)
+BuildRequires:  fdupes
 
 %description
 This is the core library of the Gtk2Hs suite of libraries for Haskell based on
@@ -75,6 +76,8 @@
 
 %install
 %ghc_lib_install
+# TODO: repack in all gtk2hs packages examples in right way
+%fdupes %{buildroot}%{_prefix}
 
 %post devel
 %ghc_pkg_recache

