Index: ghc-glib.spec
===================================================================
--- ghc-glib.spec	(revision 3)
+++ ghc-glib.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.13.2.2
 Release:        0
 Summary:        Binding to the GLIB library for Gtk2Hs
-License:        LGPL-2+
+License:        LGPL-2.0+
 Group:          System/Libraries
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
@@ -29,7 +29,6 @@
 # Begin cabal-rpm deps:
 BuildRequires:  ghc-bytestring-devel
 BuildRequires:  ghc-containers-devel
-BuildRequires:  ghc-gtk2hs-buildtools-devel
 BuildRequires:  ghc-rpm-macros
 BuildRequires:  ghc-text-devel
 BuildRequires:  ghc-utf8-string-devel

