Index: ghc-ede.spec
===================================================================
--- ghc-ede.spec	(revision 1)
+++ ghc-ede.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        0.2.8.5
 Release:        0
 Summary:        Templating language with similar syntax and features to Liquid or Jinja2
-License:        Unknown
+License:        MPL-2.0
 Group:          Development/Languages/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

