Index: ghc-HTTP.spec
===================================================================
--- ghc-HTTP.spec	(revision 7)
+++ ghc-HTTP.spec	(working copy)
@@ -26,6 +26,7 @@
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
 Source1:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/revision/2.cabal#/%{pkg_name}.cabal
+Patch0:         HTTP-4000.3.15.patch
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-array-devel
 BuildRequires:  ghc-bytestring-devel
@@ -95,7 +96,7 @@
 This package provides the Haskell %{pkg_name} library development files.
 
 %prep
-%autosetup -n %{pkg_name}-%{version}
+%autosetup -p1 -n %{pkg_name}-%{version}
 cp -p %{SOURCE1} %{pkg_name}.cabal
 cabal-tweak-dep-ver base '< 4.15' '<5'
 
