Index: ghc-HTTP.spec
===================================================================
--- ghc-HTTP.spec	(revision 2)
+++ ghc-HTTP.spec	(working copy)
@@ -26,6 +26,7 @@
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
 Source1:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/revision/1.cabal#/%{pkg_name}.cabal
+Patch1:         https://gitlab.haskell.org/ghc/head.hackage/-/raw/master/patches/HTTP-4000.3.15.patch
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-array-devel
 BuildRequires:  ghc-bytestring-devel
@@ -94,7 +95,7 @@
 This package provides the Haskell %{pkg_name} library development files.
 
 %prep
-%autosetup -n %{pkg_name}-%{version}
+%autosetup -n %{pkg_name}-%{version} -p1
 cp -p %{SOURCE1} %{pkg_name}.cabal
 cabal-tweak-dep-ver base '< 4.15' '<5'
 
