Index: texmath.spec
===================================================================
--- texmath.spec	(revision 2)
+++ texmath.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        0.8.6.4
 Release:        0
 Summary:        Conversion between formats used to represent mathematics
-License:        GPL-1.0+
+License:        GPL-2.0
 Group:          Development/Languages/Other
 Url:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
@@ -92,7 +92,7 @@
 
 %prep
 %setup -q
-
+%define cabal_configure_options -f 'executable'
 
 %build
 %ghc_lib_build
@@ -100,7 +100,7 @@
 
 %install
 %ghc_lib_install
-
+%ghc_fix_dynamic_rpath %{name}
 
 %check
 %if %{with tests}
@@ -117,7 +117,8 @@
 %files
 %defattr(-,root,root,-)
 %doc LICENSE
-%doc README.markdown changelog
+%doc README.markdown
+%{_bindir}/%{name}
 
 %files -n ghc-%{name} -f ghc-%{name}.files
 %defattr(-,root,root,-)
@@ -125,6 +126,5 @@
 
 %files -n ghc-%{name}-devel -f ghc-%{name}-devel.files
 %defattr(-,root,root,-)
-%doc README.markdown changelog
 
 %changelog

