Index: ghc-monad-coroutine.spec
===================================================================
--- ghc-monad-coroutine.spec	(revision 1)
+++ ghc-monad-coroutine.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.9.0.3
 Release:        0
 Summary:        Coroutine monad transformer for suspending and resuming monadic computations
-License:        GPL-1.0+
+License:        GPL-3.0+
 Group:          Development/Languages/Other
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

