--- ghc-webrtc-vad.spec (revision 6)
+++ ghc-webrtc-vad.spec (revision 7)
@@ -25,6 +25,8 @@
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
+# PATCH-FIX-UPSTREAM add-ppc64-typedefs.patch ptrommler@icloud.com Fix build on ppc64, ppc64le
+Patch1:         add-ppc64-typedefs.patch
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-primitive-devel
 BuildRequires:  ghc-rpm-macros
@@ -46,6 +48,7 @@
 
 %prep
 %setup -q -n %{pkg_name}-%{version}
+%patch1 -p1
 
 %build
 %ghc_lib_build

