Index: ghc-cheapskate.spec
===================================================================
--- ghc-cheapskate.spec	(revision 2)
+++ ghc-cheapskate.spec	(working copy)
@@ -41,5 +41,9 @@
 BuildRequires:  ghc-xss-sanitize-devel
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+BuildRequires:  ghc-aeson-devel
+BuildRequires:  ghc-http-types-devel
+BuildRequires:  ghc-wai-devel
+BuildRequires:  ghc-wai-extra-devel
 
 %description
 This is an experimental Markdown processor in pure Haskell. It aims to process

