Index: ghc-mongoDB.spec
===================================================================
--- ghc-mongoDB.spec	(revision 7)
+++ ghc-mongoDB.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        2.1.1
 Release:        0
 Summary:        Driver (client) for MongoDB, a free, scalable, fast, document DBMS
-License:        Unknown
+License:        Apache-2.0
 Group:          Development/Languages/Other
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

