Index: stack.spec
===================================================================
--- stack.spec	(revision 7)
+++ stack.spec	(working copy)
@@ -131,7 +131,14 @@
 %endif
 
 %description
-The Haskell Tool Stack.
+Stack is a cross-platform program for developing Haskell projects. It is aimed
+at Haskellers both new and experienced. It features:
+
+* Installing GHC automatically, in an isolated location.
+* Installing packages needed for your project.
+* Building your project.
+* Testing your project.
+* Benchmarking your project.
 
 %package -n ghc-%{name}
 Summary:        Haskell %{name} library

