Index: ghc-threepenny-gui.spec
===================================================================
--- ghc-threepenny-gui.spec	(revision 3)
+++ ghc-threepenny-gui.spec	(working copy)
@@ -82,6 +82,9 @@
 %prep
 %autosetup -n %{pkg_name}-%{version}
 cp -p %{SOURCE1} %{pkg_name}.cabal
+cabal-tweak-dep-ver 'base' '< 4.15' '< 4.16'
+cabal-tweak-dep-ver 'template-haskell' '< 2.17' '< 2.18'
+
 
 %build
 %ghc_lib_build
