Index: ghc-nix-paths.spec
===================================================================
--- ghc-nix-paths.spec	(revision 2)
+++ ghc-nix-paths.spec	(working copy)
@@ -30,10 +30,6 @@
 # Begin cabal-rpm deps:
 BuildRequires:  ghc-process-devel
 BuildRequires:  ghc-rpm-macros
-BuildRequires:  nix-build
-BuildRequires:  nix-env
-BuildRequires:  nix-instantiate
-BuildRequires:  nix-store
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 # End cabal-rpm deps
 
@@ -58,6 +54,7 @@
 
 
 %build
+%define cabal_configure_options -fallow-relative-paths
 %ghc_lib_build
 
 

