Index: ghc-math-functions.spec
===================================================================
--- ghc-math-functions.spec	(revision 1)
+++ ghc-math-functions.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        0.2.1.0
 Release:        0
 Summary:        Special functions and Chebyshev polynomials
-License:        BSD-3-Clause
+License:        BSD-2-Clause
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

