Index: ghc-Network-NineP.spec
===================================================================
--- ghc-Network-NineP.spec	(revision 3)
+++ ghc-Network-NineP.spec	(working copy)
@@ -71,6 +71,7 @@
 
 %install
 %ghc_lib_install
+rm %{buildroot}/%{_bindir}/test
 
 %post devel
 %ghc_pkg_recache
@@ -82,5 +82,4 @@
 %files -f %{name}.files
 %doc LICENSE
-%{_bindir}/test
 
 %files devel -f %{name}-devel.files

