Index: ghc-Network-NineP.spec
===================================================================
--- ghc-Network-NineP.spec	(revision 3)
+++ ghc-Network-NineP.spec	(working copy)
@@ -71,7 +71,7 @@
 
 %install
 %ghc_lib_install
-
+rm %{buildroot}/%{_bindir}/test
 
 %post devel
 %ghc_pkg_recache
@@ -82,7 +82,6 @@
 %files -f %{name}.files
 %defattr(-,root,root,-)
 %doc LICENSE
-%{_bindir}/test
 
 %files devel -f %{name}-devel.files
 %defattr(-,root,root,-)

