Index: ghc-direct-sqlite.spec
===================================================================
--- ghc-direct-sqlite.spec	(revision 1)
+++ ghc-direct-sqlite.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        2.3.19
 Release:        0
 Summary:        Low-level binding to SQLite3.  Includes UTF8 and BLOB support
-License:        BSD-3-Clause
+License:        MIT
 Group:          Development/Languages/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

