Index: cabal-install.spec
===================================================================
--- cabal-install.spec	(revision 6)
+++ cabal-install.spec	(working copy)
@@ -132,6 +132,7 @@
 BuildRequires:  ghc-tree-diff-devel
 BuildRequires:  ghc-tree-diff-prof
 %endif
+Patch01: Cabal-absolute-datadir.patch
 
 %description
 The 'cabal' command-line program simplifies the process of managing Haskell
@@ -171,7 +172,7 @@
 This package provides the Haskell %{pkg_name} profiling library.
 
 %prep
-%autosetup
+%autosetup -p1
 cp -p %{SOURCE1} %{name}.cabal
 
 %build
