Index: ghc-stm-extras.spec
===================================================================
--- ghc-stm-extras.spec	(revision 3)
+++ ghc-stm-extras.spec	(working copy)
@@ -31,6 +31,7 @@
 
 %description
-Extra STM functions.
+A collection of useful functions for dealing with Software Transactional Memory
+in Haskell. This library extends the "stm" core library.
 
 %package devel
 Summary:        Haskell %{pkg_name} library development files

