Index: ghc-recursion-schemes.spec
===================================================================
--- ghc-recursion-schemes.spec	(revision 2)
+++ ghc-recursion-schemes.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        4.1.2
 Release:        0
 Summary:        Generalized bananas, lenses and barbed wire
-License:        BSD-3-Clause
+License:        BSD-2-Clause
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

