Index: c2hs.spec
===================================================================
--- c2hs.spec	(revision 3)
+++ c2hs.spec	(working copy)
@@ -25,7 +25,6 @@
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
-BuildRequires:  c2hs
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-array-devel
 BuildRequires:  ghc-bytestring-devel

