Index: ghc-interpolatedstring-perl6.spec
===================================================================
--- ghc-interpolatedstring-perl6.spec	(revision 1)
+++ ghc-interpolatedstring-perl6.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        1.0.0
 Release:        0
 Summary:        QuasiQuoter for Perl6-style multi-line interpolated strings
-License:        SUSE-Public-Domain
+License:        CC0-1.0
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

