Index: gtk2hs-buildtools.spec
===================================================================
--- gtk2hs-buildtools.spec	(revision 6)
+++ gtk2hs-buildtools.spec	(working copy)
@@ -62,6 +62,7 @@
 Requires:       ghc-compiler = %{ghc_version}
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
+Requires:       %{name} = %{version}-%{release}
 
 %description -n ghc-%{name}-devel
 This package provides the Haskell %{name} library development files.

