Index: happy.spec
===================================================================
--- happy.spec	(revision 37)
+++ happy.spec	(working copy)
@@ -25,12 +25,17 @@
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
 Source1:        https://hackage.haskell.org/package/%{name}-%{version}/revision/1.cabal#/%{name}.cabal
+BuildRequires:  autoconf
+BuildRequires:  docbook-dtd
+BuildRequires:  docbook-xsl-stylesheets
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-array-devel
 BuildRequires:  ghc-containers-devel
 BuildRequires:  ghc-mtl-devel
 BuildRequires:  ghc-rpm-macros
 ExcludeArch:    %{ix86}
+BuildRequires:  libxml2
+BuildRequires:  libxslt
 %if %{with tests}
 BuildRequires:  ghc-process-devel
 %endif
@@ -46,9 +51,14 @@
 %build
 %define cabal_configure_options -f-bootstrap
 %ghc_bin_build
+cd doc
+autoreconf
+%configure
+%make_build html
 
 %install
 %ghc_bin_install
+install -D --mode=444 doc/%{name}.1 %{buildroot}%{_mandir}/man1/%{name}.1
 
 %check
 %cabal_test
@@ -58,6 +68,7 @@
 %doc ChangeLog.md README.md doc examples
 %{_bindir}/%{name}
 %dir %{_datadir}/%{name}-%{version}
+%{_mandir}/man1/*
 %{_datadir}/%{name}-%{version}/GLR_Base.hs
 %{_datadir}/%{name}-%{version}/GLR_Lib.hs
 %{_datadir}/%{name}-%{version}/HappyTemplate.hs
