Index: happy.spec
===================================================================
--- happy.spec	(revision 7)
+++ happy.spec	(working copy)
@@ -41,6 +41,11 @@
 BuildRequires:  ghc-process-devel
 BuildRequires:  ghc-process-prof
 %endif
+BuildRequires:  autoconf
+BuildRequires:  docbook-dtd
+BuildRequires:  docbook-xsl-stylesheets
+BuildRequires:  libxml2
+BuildRequires:  libxslt
 
 %description
 Happy is a parser generator for Haskell. Given a grammar specification in BNF,
@@ -53,9 +58,14 @@
 %build
 %define cabal_configure_options -f-bootstrap
 %ghc_bin_build
+cd doc
+autoreconf
+%configure
+%make_build html
 
 %install
 %ghc_bin_install
+install -D --mode=444 doc/%{name}.1 %{buildroot}%{_mandir}/man1/%{name}.1
 
 %check
 %cabal_test
@@ -65,4 +75,5 @@
 %doc ChangeLog.md README.md examples
 %{_bindir}/%{name}
 
+%{_mandir}/man1/*
 %changelog
