Index: happy.spec
===================================================================
--- happy.spec	(revision 38)
+++ happy.spec	(working copy)
@@ -58,6 +58,9 @@
 
 %install
 %ghc_bin_install
+# drop artifacts from autoconf that differ across builds to fix build-compare
+rm -rf doc/autom4te.cache doc/config.log doc/config.status
+ 
 install -D --mode=444 doc/%{name}.1 %{buildroot}%{_mandir}/man1/%{name}.1
 
 %check
