Index: happy.spec
===================================================================
--- happy.spec	(revision 15)
+++ happy.spec	(working copy)
@@ -62,6 +62,8 @@
 %install
 %ghc_bin_install
 install -D --mode=444 doc/%{name}.1 %{buildroot}%{_mandir}/man1/%{name}.1
+# drop artifacts from autoconf that differ across builds to fix build-compare
+rm -rf doc/autom4te.cache doc/config.log doc/config.status
 
 %check
 %cabal_test
