Index: happy.spec
===================================================================
--- happy.spec	(revision 8)
+++ happy.spec	(working copy)
@@ -30,6 +30,11 @@
 BuildRequires:  ghc-containers-devel
 BuildRequires:  ghc-mtl-devel
 BuildRequires:  ghc-rpm-macros
+BuildRequires:  autoconf
+BuildRequires:  docbook-dtd
+BuildRequires:  docbook-xsl-stylesheets
+BuildRequires:  libxml2
+BuildRequires:  libxslt
 %if %{with tests}
 BuildRequires:  ghc-process-devel
 %endif
@@ -44,9 +49,14 @@
 
 %build
 %ghc_bin_build
+cd doc
+autoreconf
+./configure
+make %{?_smp_mflags} html
 
 %install
 %ghc_bin_install
+install -D --mode=444 doc/%{name}.1 %{buildroot}%{_mandir}/man1/%{name}.1
 
 %check
 %cabal_test
@@ -56,6 +66,7 @@
 %doc CHANGES README.md TODO doc examples
 %{_bindir}/%{name}
 %dir %{_datadir}/%{name}-%{version}
+%{_mandir}/man1/*
 %{_datadir}/%{name}-%{version}/GLR_Base
 %{_datadir}/%{name}-%{version}/GLR_Lib
 %{_datadir}/%{name}-%{version}/GLR_Lib-ghc

