Index: happy.spec
===================================================================
--- happy.spec	(revision 5)
+++ happy.spec	(working copy)
@@ -21,11 +21,16 @@
 Version:        1.19.5
 Release:        0
 Summary:        Happy is a parser generator for Haskell
-License:        BSD-3-Clause
-Group:          Development/Languages/Other
+License:        BSD-2-Clause
+Group:          Development/Languages/Other
 Url:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
 Source1:        https://hackage.haskell.org/package/%{name}-%{version}/revision/1.cabal
 BuildRequires:  ghc-Cabal-devel
+BuildRequires:  autoconf
+BuildRequires:  docbook-dtd
+BuildRequires:  docbook-xsl-stylesheets
+BuildRequires:  libxml2
+BuildRequires:  libxslt
 BuildRequires:  ghc-array-devel
 BuildRequires:  ghc-containers-devel
@@ -50,8 +55,13 @@
 
 %build
 %ghc_bin_build
+cd doc
+autoreconf
+%configure
+make %{?_smp_mflags} html
 
 %install
 %ghc_bin_install
+install -D --mode=444 doc/%{name}.1 %{buildroot}%{_mandir}/man1/%{name}.1
 
 %check
@@ -67,5 +76,7 @@
 %doc LICENSE
 %doc README TODO doc examples
 %{_bindir}/%{name}
+%{_datadir}/%{name}-%{version}
+%{_mandir}/man1/*
 
 %changelog

