Index: ghc-FenwickTree.spec
===================================================================
--- ghc-FenwickTree.spec	(revision 2)
+++ ghc-FenwickTree.spec	(working copy)
@@ -78,9 +78,10 @@
 %files -f %{name}.files
 %defattr(-,root,root,-)
 %doc LICENSE
+%dir %{_datadir}/%{pkg_name}-%{version}
+%{_datadir}/%{pkg_name}-%{version}/*
 
 %files devel -f %{name}-devel.files
 %defattr(-,root,root,-)
-%doc README.md changelog
 
 %changelog

