Index: ghc-htaglib.spec
===================================================================
--- ghc-htaglib.spec	(revision 2)
+++ ghc-htaglib.spec	(working copy)
@@ -31,6 +31,7 @@
 BuildRequires:  ghc-bytestring-devel
 BuildRequires:  ghc-rpm-macros
 BuildRequires:  ghc-text-devel
+BuildRequires:  libtag-devel
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 %if %{with tests}
 BuildRequires:  ghc-HUnit-devel

