Index: ghc-countable.spec
===================================================================
--- ghc-countable.spec	(revision 1)
+++ ghc-countable.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        1.0
 Release:        0
 Summary:        Countable, Searchable, Finite, Empty classes
-License:        BSD-3-Clause
+License:        BSD-2-Clause
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

