Index: pandoc.spec
===================================================================
--- pandoc.spec	(revision 16)
+++ pandoc.spec	(working copy)
@@ -28,5 +28,6 @@
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
 Source1:        https://hackage.haskell.org/package/%{name}-%{version}/revision/2.cabal#/%{name}.cabal
+Source10:       pandoc-pdf.readme
 BuildRequires:  chrpath
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-Glob-devel
@@ -110,6 +111,19 @@
 writers, which convert this native representation into a target format.
 Thus, adding an input or output format requires only adding a reader or writer.
 
+%if 0%{?is_opensuse}
+%package pdf
+Summary:        Meta package for support generating pdf with pandoc
+Group:          Productivity/Publishing/PDF
+Requires:       %{name} = %{version}
+Requires:       texlive-latex-bin-bin
+# bsc#1014590
+Requires:       texlive-upquote
+
+%description pdf
+Meta package for support generating pdf with pandoc.
+%endif
+
 %package -n ghc-%{name}
 Summary:        Haskell %{name} library
 Group:          System/Libraries
@@ -130,6 +144,7 @@
 
 %prep
 %setup -q
+cp %{SOURCE10} .
 cp -p %{SOURCE1} %{name}.cabal
 
 %build
@@ -304,6 +319,11 @@
 %{_datadir}/%{name}-%{version}/data/templates/default.zimwiki
 %{_datadir}/%{name}-%{version}/data/translations/*.yaml
 
+%if 0%{?is_opensuse}
+%files pdf
+%doc pandoc-pdf.readme
+%endif
+
 %files -n ghc-%{name} -f ghc-%{name}.files
 %license COPYING.md
 

