Index: ghc-gd.spec
===================================================================
--- ghc-gd.spec	(revision 2)
+++ ghc-gd.spec	(working copy)
@@ -60,6 +60,7 @@
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
 # End cabal-rpm deps
+Requires:  gd-devel
 
 %description devel
 This package provides the Haskell %{pkg_name} library development files.

