Index: ghc-cabal-helper.spec
===================================================================
--- ghc-cabal-helper.spec	(revision 8)
+++ ghc-cabal-helper.spec	(working copy)
@@ -88,6 +88,7 @@
 
 
 %build
+%define cabal_configure_options --libexecdir=%{_libexecdir}
 %ghc_lib_build
 
 
@@ -110,7 +111,7 @@
 %files -f %{name}.files
 %defattr(-,root,root,-)
 %doc LICENSE
-%{_bindir}/cabal-helper-wrapper
+%{_libexecdir}/cabal-helper-wrapper
 
 %files devel -f %{name}-devel.files
 %defattr(-,root,root,-)

