Index: ghc-aws.spec
===================================================================
--- ghc-aws.spec	(revision 2)
+++ ghc-aws.spec	(working copy)
@@ -26,6 +26,7 @@
 License:        BSD-3-Clause
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
+Patch01:        0001-switch-from-memory-or-ram-to-allow-building-with-htt.patch
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-aeson-devel
 BuildRequires:  ghc-aeson-prof
