Index: ghc-tasty-auto.spec
===================================================================
--- ghc-tasty-auto.spec (revision 2)
+++ ghc-tasty-auto.spec (revision 3)
@@ -31,6 +31,7 @@
 BuildRequires:  ghc-directory-devel
 BuildRequires:  ghc-filepath-devel
 BuildRequires:  ghc-rpm-macros
+BuildRequires:  git-core
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 %if %{with tests}
 BuildRequires:  ghc-tasty-devel

