Index: cabal2spec.spec
===================================================================
--- cabal2spec.spec	(revision 2)
+++ cabal2spec.spec	(working copy)
@@ -35,6 +35,7 @@
 BuildRequires:  ghc-tasty-devel
 BuildRequires:  ghc-tasty-golden-devel
 %endif
+Patch01:        0001-Update-code-for-Cabal-3.2.x.patch
 
 %description
 Convert Cabal files into a
@@ -63,6 +64,7 @@
 
 %prep
 %setup -q
+%patch01 -p1
 
 %build
 %ghc_lib_build
