Index: cabal-install.spec
===================================================================
--- cabal-install.spec	(revision 32)
+++ cabal-install.spec	(working copy)
@@ -23,6 +23,8 @@
 License:        BSD-3-Clause
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
+Patch01:        ghc-8.10-support-for-3.2.patch
+Patch02:        base16-bytestring.patch
 Source1:        https://hackage.haskell.org/package/%{name}-%{version}/revision/1.cabal#/%{name}.cabal
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-HTTP-devel
@@ -64,7 +66,7 @@
 installation of Haskell libraries and programs.
 
 %prep
-%autosetup
+%autosetup -p2
 cp -p %{SOURCE1} %{name}.cabal
 
 %build
