Index: ghc-eventful-sql-common.spec
===================================================================
--- ghc-eventful-sql-common.spec	(revision 1)
+++ ghc-eventful-sql-common.spec	(working copy)
@@ -54,7 +54,9 @@
 
 %prep
 %setup -q -n %{pkg_name}-%{version}
 cp -p %{SOURCE1} %{pkg_name}.cabal
+echo > Setup.hs 'import Distribution.Simple'
+echo >>Setup.hs 'main = defaultMain'
 
 %build
 %ghc_lib_build

