Index: shake.spec
===================================================================
--- shake.spec	(revision 5)
+++ shake.spec	(working copy)
@@ -99,6 +99,7 @@
 
 %install
 %ghc_lib_install
+rm -Rf %{buildroot}/%{_datadir}/%{name}-%{version}/docs
 
 %check
 %if %{with tests}
@@ -120,17 +120,9 @@
 %{_bindir}/%{name}
 %dir %{_datadir}/%{name}-%{version}
 %dir %{_datadir}/%{name}-%{version}/html
-%dir %{_datadir}/%{name}-%{version}/docs
-%dir %{_datadir}/%{name}-%{version}/docs/manual
 %{_datadir}/%{name}-%{version}/html/viz.js
 %{_datadir}/%{name}-%{version}/html/profile.html
 %{_datadir}/%{name}-%{version}/html/progress.html
 %{_datadir}/%{name}-%{version}/html/shake.js
-%{_datadir}/%{name}-%{version}/docs/manual/build.bat
-%{_datadir}/%{name}-%{version}/docs/manual/Build.hs
-%{_datadir}/%{name}-%{version}/docs/manual/build.sh
-%{_datadir}/%{name}-%{version}/docs/manual/constants.c
-%{_datadir}/%{name}-%{version}/docs/manual/constants.h
-%{_datadir}/%{name}-%{version}/docs/manual/main.c
 
 %files -n ghc-%{name} -f ghc-%{name}.files

