Index: ghc-engine-io-wai.spec
===================================================================
--- ghc-engine-io-wai.spec	(revision 2)
+++ ghc-engine-io-wai.spec	(working copy)
@@ -20,7 +20,7 @@
 Name:           ghc-%{pkg_name}
 Version:        1.0.6
 Release:        0
-Summary:        FIXME
+Summary:        An 'engine-io' server API implementation for WAI
 License:        BSD-3-Clause
 Group:          Development/Languages/Other
 Url:            https://hackage.haskell.org/package/%{pkg_name}
@@ -43,8 +43,8 @@
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 
 %description
-This package provides an 'engine-io' 'ServerAPI' that is compatible with
-<https://hackage.haskell.org/package/wai/ Wai>.
+This package provides an 'engine-io' server API that is compatible with Wai
+<https://hackage.haskell.org/package/wai/>.
 
 %package devel
 Summary:        Haskell %{pkg_name} library development files

