Index: cabal-install.spec
===================================================================
--- cabal-install.spec	(revision 3)
+++ cabal-install.spec	(working copy)
@@ -50,6 +50,7 @@
 BuildRequires:  ghc-resolv-devel
 BuildRequires:  ghc-rpm-macros
 BuildRequires:  ghc-stm-devel
+Suggests:       %{name}-bash-completion
 BuildRequires:  ghc-tar-devel
 BuildRequires:  ghc-time-devel
 BuildRequires:  ghc-unix-devel
@@ -63,6 +64,16 @@
 This package only provides an executable and cannot be used as a library
 (ignore the module listing below.).
 
+%package bash-completion
+Summary:        Bash completion for cabal-install
+Group:          System/Shells
+Requires:       %{name} = %{version}
+Requires:       bash-completion
+Supplements:    packageand(%{name}:bash-completion)
+
+%description bash-completion
+Optional sub-package offering bash completion for cabal-install.
+
 %prep
 %setup -q
 cp -p %{SOURCE1} %{name}.cabal
@@ -72,6 +83,7 @@
 
 %install
 %ghc_bin_install
+install -D -m444 bash-completion/cabal %{buildroot}%{_datadir}/bash-completion/completions/cabal
 
 %files
 %license LICENSE
@@ -79,4 +91,9 @@
 %{_bindir}/cabal
 %{_mandir}/man1/cabal.1%{?ext_man}
 
+%files bash-completion
+%dir %{_datadir}/bash-completion
+%dir %{_datadir}/bash-completion/completions
+%{_datadir}/bash-completion/completions/cabal
+
 %changelog

