Index: ghc-hs-bibutils.spec
===================================================================
--- ghc-hs-bibutils.spec	(revision 1)
+++ ghc-hs-bibutils.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        5.5
 Release:        0
 Summary:        Haskell bindings to bibutils, the bibliography conversion utilities
-License:        GPL-1.0-or-later
+License:        GPL-2.0-or-later
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

