Index: ghc-libnotify.spec
===================================================================
--- ghc-libnotify.spec	(revision 1)
+++ ghc-libnotify.spec	(working copy)
@@ -33,6 +33,7 @@
 BuildRequires:  ghc-rpm-macros
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 # End cabal-rpm deps
+BuildRequires:  libnotify-devel
 
 %description
 The package provides a high level interface to libnotify API (see "Libnotify")

