Index: ghc-libnotify.spec
===================================================================
--- ghc-libnotify.spec	(revision 2)
+++ ghc-libnotify.spec	(working copy)
@@ -48,6 +48,7 @@
 Requires:       ghc-compiler = %{ghc_version}
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
+Requires:       libnotify-devel
 
 %description devel
 This package provides the Haskell %{pkg_name} library development files.

