Index: ghc-bioalign.spec
===================================================================
--- ghc-bioalign.spec	(revision 2)
+++ ghc-bioalign.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.0.5
 Release:        0
 Summary:        Data structures and helper functions for calculating alignments
-License:        GPL-1.0+
+License:        GPL-3.0
 Group:          Development/Languages/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

