Index: ghc-yesod-core.spec
===================================================================
--- ghc-yesod-core.spec	(revision 1)
+++ ghc-yesod-core.spec	(working copy)
@@ -26,8 +26,8 @@
 License:        MIT
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
+Patch1:         drop-dependency-on-empty-attoparsec-aeson-library.patch
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-aeson-devel
-BuildRequires:  ghc-attoparsec-aeson-devel
 BuildRequires:  ghc-auto-update-devel
 BuildRequires:  ghc-base-devel
@@ -155,7 +154,7 @@
 This package provides the Haskell %{pkg_name} profiling library.
 
 %prep
-%autosetup -n %{pkg_name}-%{version}
+%autosetup -n %{pkg_name}-%{version} -p1
 
 %build
 %ghc_lib_build
