From a1c2d2d45eaa74f6be6baf201c590a9397a671b5 Mon Sep 17 00:00:00 2001
From: Peter Simons <simons@cryp.to>
Date: Wed, 20 Jul 2016 21:24:13 +0200
Subject: [PATCH 2/2] de-duplicate the installation

---
 ghc-pango.spec | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/ghc-pango.spec b/ghc-pango.spec
index ff249d3..1006712 100644
--- a/ghc-pango.spec
+++ b/ghc-pango.spec
@@ -37,6 +37,7 @@ BuildRequires:  ghc-process-devel
 BuildRequires:  ghc-rpm-macros
 BuildRequires:  ghc-text-devel
 BuildRequires:  gtk2hs-buildtools
+BuildRequires:  fdupes
 BuildRequires:  pkgconfig
 BuildRequires:  pkgconfig(cairo)
 BuildRequires:  pkgconfig(pango)
@@ -74,6 +75,7 @@ This package provides the Haskell %{pkg_name} library development files.
 
 %install
 %ghc_lib_install
+%fdupes %{buildroot}%{_prefix}
 
 
 %post devel
-- 
2.9.0

