--- ghc-hsignal.spec (revision 6)
+++ ghc-hsignal.spec (revision 7)
@@ -25,6 +25,8 @@
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
+# PATCH-FIX-UPSTREAM fix-gcc-flags.patch ptrommler@icloud.com -- Do not pass -msse2 to gcc on non-Intel/AMD arches
+Patch:          fix-gcc-flags.patch
 BuildRequires:  blas-devel
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-array-devel
@@ -69,6 +71,7 @@
 
 %prep
 %setup -q -n %{pkg_name}-%{version}
+%patch -p1
 
 %build
 %ghc_lib_build

