Index: cabal-rpm.spec
===================================================================
--- cabal-rpm.spec	(revision 5)
+++ cabal-rpm.spec	(working copy)
@@ -59,10 +59,12 @@
 
 %install
 %ghc_bin_install
+install -p -m 0644 -D man/cabal-rpm.1 %{buildroot}%{_mandir}/man1/cabal-rpm.1
 
 %files
 %license COPYING
 %doc ChangeLog README.md TODO
 %{_bindir}/%{name}
+%{_mandir}/man1/cabal-rpm.1*
 
 %changelog

