Index: cabal-rpm.spec
===================================================================
--- cabal-rpm.spec	(revision 2)
+++ cabal-rpm.spec	(working copy)
@@ -21,7 +21,7 @@
 Release:        0
 Summary:        RPM packaging tool for Haskell Cabal-based packages
 License:        GPL-3.0+
-Group:          System/Libraries
+Group:          Development/Tools/Other
 Url:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
 BuildRequires:  ghc-Cabal-devel
@@ -33,5 +33,8 @@
 BuildRequires:  ghc-rpm-macros
 BuildRequires:  ghc-time-devel
 BuildRequires:  ghc-unix-devel
+Recommends:     cabal-install
+Recommends:     ghc
+Recommends:     yum-utils
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 
@@ -53,6 +56,8 @@
 %build
 %ghc_bin_build
 
+install -p -m 0644 -D man/cblrpm.1 %{buildroot}%{_mandir}/man1/cblrpm.1
+
 
 %install
 %ghc_bin_install
@@ -63,5 +68,6 @@
 %doc COPYING
 %doc ChangeLog README.md
 %{_bindir}/cblrpm
+%{_mandir}/man1/cblrpm.1*
 
 %changelog

