Index: ghc-nested-routes.spec
===================================================================
--- ghc-nested-routes.spec	(revision 7)
+++ ghc-nested-routes.spec	(working copy)
@@ -26,7 +26,6 @@
 Group:          Development/Languages/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
-BuildRequires:  chrpath
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-attoparsec-devel
 BuildRequires:  ghc-bytestring-devel
@@ -84,10 +83,9 @@
 
 %build
-%define cabal_configure_options -fexample
+%define cabal_configure_options -f-example
 %ghc_lib_build
 
 %install
 %ghc_lib_install
-%ghc_fix_rpath %{pkg_name}-%{version}
 
 %check
@@ -106,5 +104,4 @@
 %files -f %{name}.files
 %doc LICENSE
-%{_bindir}/example
 
 %files devel -f %{name}-devel.files

