Index: ghc-array-memoize.spec
===================================================================
--- ghc-array-memoize.spec	(revision 1)
+++ ghc-array-memoize.spec	(working copy)
@@ -21,6 +21,6 @@
 Version:        0.6.0
 Release:        0
 Summary:        Memoization combinators using arrays for finite sub-domains of functions
-License:        BSD-3-Clause
+License:        BSD-2-Clause
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

