Index: ghc-postgresql-simple-migration.spec
===================================================================
--- ghc-postgresql-simple-migration.spec	(revision 5)
+++ ghc-postgresql-simple-migration.spec	(working copy)
@@ -38,6 +38,8 @@
 %if %{with tests}
 BuildRequires:  ghc-hspec-devel
 %endif
+Requires(post): update-alternatives
+Requires(postun): update-alternatives
 
 %description
 A PostgreSQL-simple schema migration utility.
@@ -63,8 +65,22 @@
 %install
 %ghc_lib_install
 
+# create a dummy target for /etc/alternatives/migrate
+mkdir -p %{buildroot}%{_sysconfdir}/alternatives
+mv %{buildroot}%{_bindir}/migrate %{buildroot}%{_bindir}/migrate-ghc
+ln -s -f %{_sysconfdir}/alternatives/migrate %{buildroot}%{_bindir}/migrate
+
 %check
 %cabal_test
+
+%post
+update-alternatives --install \
+   %{_bindir}/migrate migrate %{_bindir}/migrate-ghc 15
+
+%postun
+if [ ! -f %{_bindir}/migrate-ghc ] ; then
+   update-alternatives --remove migrate %{_bindir}/migrate-ghc
+fi
 
 %post devel
 %ghc_pkg_recache
@@ -75,6 +91,8 @@
 %files -f %{name}.files
 %doc License
 %{_bindir}/migrate
+%{_bindir}/migrate-ghc
+%ghost %_sysconfdir/alternatives/migrate
 
 %files devel -f %{name}-devel.files
 %doc Changelog.markdown Readme.markdown

