Index: ghc-persistent-template.spec
===================================================================
--- ghc-persistent-template.spec	(revision 9)
+++ ghc-persistent-template.spec	(working copy)
@@ -26,6 +26,7 @@
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
 Source1:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/revision/1.cabal#/%{pkg_name}.cabal
+Patch0:         pt-ghc9.patch
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-aeson-devel
 BuildRequires:  ghc-bytestring-devel
@@ -63,8 +64,9 @@
 files.
 
 %prep
-%autosetup -n %{pkg_name}-%{version}
+%autosetup -p1 -n %{pkg_name}-%{version}
 cp -p %{SOURCE1} %{pkg_name}.cabal
+cabal-tweak-dep-ver "template-haskell" '< 2.17' '< 2.18'
 
 %build
 %ghc_lib_build
