Index: ghc-speculation.spec
===================================================================
--- ghc-speculation.spec	(revision 1)
+++ ghc-speculation.spec	(working copy)
@@ -21,6 +21,6 @@
 Version:        1.5.0.3
 Release:        0
 Summary:        A framework for safe, programmable, speculative parallelism
-License:        BSD-3-Clause
+License:        BSD-2-Clause
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

