Index: ghc-s3-signer.spec
===================================================================
--- ghc-s3-signer.spec	(revision 1)
+++ ghc-s3-signer.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.3.0.0
 Release:        0
 Summary:        Pre-signed Amazon S3 URLs
-License:        BSD-3-Clause
+License:        BSD-2-Clause
 Group:          Development/Languages/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

