Index: pandoc.spec
===================================================================
--- pandoc.spec	(revision 11)
+++ pandoc.spec	(working copy)
@@ -25,6 +25,7 @@
 License:        GPL-2.0-or-later
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
+Patch0:         pandoc-2.11.0.4.patch
 BuildRequires:  chrpath
 BuildRequires:  fdupes
 BuildRequires:  ghc-Cabal-devel
@@ -141,7 +142,7 @@
 This package provides the Haskell %{name} library development files.
 
 %prep
-%autosetup
+%autosetup -p1
 cabal-tweak-dep-ver base64-bytestring '< 1.2' '< 2'
 cabal-tweak-dep-ver random '< 1.2' '< 1.3'
 
