Index: ghc-riak-protobuf.spec
===================================================================
--- ghc-riak-protobuf.spec	(revision 5)
+++ ghc-riak-protobuf.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.21.0.0
 Release:        0
 Summary:        Haskell types for the Riak protocol buffer API
-License:        Unknown
+License:        Apache-2.0
 Group:          Development/Languages/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

