Index: cabal-rpm.spec
===================================================================
--- cabal-rpm.spec	(revision 8)
+++ cabal-rpm.spec	(working copy)
@@ -55,11 +55,13 @@
 
 %install
 %ghc_bin_install
+install -p -m 0644 -D man/cabal-rpm.1 %{buildroot}%{_mandir}/man1/cabal-rpm.1
 
 %files
 %defattr(-,root,root,-)
 %doc COPYING
 %doc ChangeLog README.md
 %{_bindir}/%{name}
+%{_mandir}/man1/cabal-rpm.1*
 
 %changelog

