Index: ghc-lens.spec
===================================================================
--- ghc-lens.spec	(revision 1)
+++ ghc-lens.spec	(working copy)
@@ -71,6 +71,7 @@
 BuildRequires:  ghc-test-framework-quickcheck2-devel
 BuildRequires:  ghc-test-framework-th-devel
 %endif
+Patch01:        fix-ghc-8.10.x.patch
 
 %description
 This package comes "Batteries Included" with many useful lenses for the types
@@ -168,6 +169,8 @@
 
 %prep
 %setup -q -n %{pkg_name}-%{version}
+%patch01 -p1
+cabal-tweak-dep-ver template-haskell '< 2.16' '< 3'
 
 %build
 %ghc_lib_build
