Index: ghc-hsdev.spec
===================================================================
--- ghc-hsdev.spec	(revision 1)
+++ ghc-hsdev.spec	(working copy)
@@ -35,7 +35,6 @@
 BuildRequires:  ghc-array-devel
 BuildRequires:  ghc-async-devel
 BuildRequires:  ghc-attoparsec-devel
-BuildRequires:  ghc-bin-package-db-devel
 BuildRequires:  ghc-bytestring-devel
 BuildRequires:  ghc-containers-devel
 BuildRequires:  ghc-cpphs-devel

