Index: ghc-api-field-json-th.spec
===================================================================
--- ghc-api-field-json-th.spec	(revision 5)
+++ ghc-api-field-json-th.spec	(working copy)
@@ -22,6 +22,6 @@
 Version:        0.1.0.1
 Release:        0
 Summary:        Option of aeson's deriveJSON
-License:        BSD-3-Clause
+License:        MIT
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

