Index: ghc-rss-conduit.spec
===================================================================
--- ghc-rss-conduit.spec	(revision 4)
+++ ghc-rss-conduit.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        0.3.0.0
 Release:        0
 Summary:        Streaming parser/renderer for the RSS 2.0 standard
-License:        SUSE-Public-Domain
+License:        WTFPL
 Group:          Development/Languages/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

