Index: ghc-pandoc-citeproc.spec
===================================================================
--- ghc-pandoc-citeproc.spec	(revision 13)
+++ ghc-pandoc-citeproc.spec	(working copy)
@@ -90,10 +90,10 @@
 
 %build
-%define cabal_configure_options -ftest_citeproc
+%define cabal_configure_options -f-test_citeproc
 %ghc_lib_build
 
 %install
 %ghc_lib_install
-%ghc_fix_rpath %{pkg_name}-%{version}
+%ghc_fix_rpath %{pkg_name}-%{version}
 
 %check
@@ -113,7 +113,6 @@
 %defattr(-,root,root,-)
 %doc LICENSE
 %{_bindir}/%{pkg_name}
-%{_bindir}/test-citeproc
 %dir %{_datadir}/%{pkg_name}-%{version}
 %dir %{_datadir}/%{pkg_name}-%{version}/locales
 %dir %{_datadir}/%{pkg_name}-%{version}/man

