Index: ghc-secp256k1.spec
===================================================================
--- ghc-secp256k1.spec	(revision 4)
+++ ghc-secp256k1.spec	(working copy)
@@ -22,6 +22,6 @@
 Version:        0.4.8
 Release:        0
 Summary:        Bindings for secp256k1 library from Bitcoin Core
-License:        SUSE-Public-Domain
+License:        Unlicense
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

