Index: ghc-open-browser.spec
===================================================================
--- ghc-open-browser.spec	(revision 4)
+++ ghc-open-browser.spec	(working copy)
@@ -58,7 +58,7 @@
 
 %install
 %ghc_lib_install
-%ghc_fix_rpath %{pkg_name}-%{version}
+rm %{buildroot}/%{_bindir}/example
 
 
 %post devel
@@ -71,7 +70,6 @@
 %files -f %{name}.files
 %defattr(-,root,root,-)
 %doc LICENSE
-%{_bindir}/example
 
 %files devel -f %{name}-devel.files
 %defattr(-,root,root,-)

