Index: ghc-syb.spec
===================================================================
--- ghc-syb.spec	(revision 1)
+++ ghc-syb.spec	(working copy)
@@ -25,6 +25,7 @@
 License:        BSD-3-Clause
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
+Patch0:         syb-0.7.1.patch
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-rpm-macros
 %if %{with tests}
@@ -51,7 +52,7 @@
 This package provides the Haskell %{pkg_name} library development files.
 
 %prep
-%autosetup -n %{pkg_name}-%{version}
+%autosetup -p1 -n %{pkg_name}-%{version}
 
 %build
 %ghc_lib_build
