Index: ghc-graylog.spec
===================================================================
--- ghc-graylog.spec	(revision 5)
+++ ghc-graylog.spec	(working copy)
@@ -22,6 +22,6 @@
 Version:        0.1.0.1
 Release:        0
 Summary:        Support for graylog output
-License:        Unknown
+License:        WTFPL
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

