Index: ghc-cryptonite.spec
===================================================================
--- ghc-cryptonite.spec	(revision 1)
+++ ghc-cryptonite.spec	(working copy)
@@ -25,6 +25,7 @@
 License:        BSD-3-Clause
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
+Patch0:         cryptonite-0.27.patch
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-basement-devel
 BuildRequires:  ghc-bytestring-devel
@@ -80,7 +81,7 @@
 This package provides the Haskell %{pkg_name} library development files.
 
 %prep
-%autosetup -n %{pkg_name}-%{version}
+%autosetup -p1 -n %{pkg_name}-%{version}
 
 %build
 %define cabal_configure_options -f-support_aesni -f-support_blake2_sse -f-support_rdrand
