From 4b50150ded02113c9ed0378f35b4f81a5323f9ce Mon Sep 17 00:00:00 2001
From: Peter Simons <simons@cryp.to>
Date: Wed, 20 Jul 2016 21:32:36 +0200
Subject: [PATCH 2/2] de-duplicate the installation

---
 ghc-gtk3.spec | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/ghc-gtk3.spec b/ghc-gtk3.spec
index c6262de..a9f3bcb 100644
--- a/ghc-gtk3.spec
+++ b/ghc-gtk3.spec
@@ -39,6 +39,7 @@ BuildRequires:  ghc-pango-devel
 BuildRequires:  ghc-rpm-macros
 BuildRequires:  ghc-text-devel
 BuildRequires:  gtk2hs-buildtools
+BuildRequires:  fdupes
 BuildRequires:  pkgconfig
 BuildRequires:  pkgconfig(gthread-2.0)
 BuildRequires:  pkgconfig(gtk+-3.0)
@@ -74,6 +75,8 @@ This package provides the Haskell %{pkg_name} library development files.
 
 %install
 %ghc_lib_install
+# TODO: repack in all gtk2hs packages examples in right way
+%fdupes %{buildroot}%{_prefix}
 
 
 %post devel
-- 
2.9.0

