Index: ghc-gtk3.spec
===================================================================
--- ghc-gtk3.spec	(revision 24)
+++ ghc-gtk3.spec	(working copy)
@@ -25,7 +25,6 @@
 Group:          Development/Languages/Other
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
-BuildRequires:  chrpath
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-array-devel
 BuildRequires:  ghc-bytestring-devel
@@ -69,10 +68,9 @@
 
 %build
-%define cabal_configure_options -fbuild-demos
+%define cabal_configure_options -f-build-demos
 %ghc_lib_build
 
 %install
 %ghc_lib_install
-%{ghc_fix_rpath} %{pkg_name}-%{version}
 
 %post devel
@@ -87,22 +85,6 @@
 %files -f %{name}.files
 %defattr(-,root,root,-)
 %doc COPYING
-%{_bindir}/gtk2hs-demo-actionMenu
-%{_bindir}/gtk2hs-demo-arabic
-%{_bindir}/gtk2hs-demo-builder
-%{_bindir}/gtk2hs-demo-buttonBox
-%{_bindir}/gtk2hs-demo-carsim
-%{_bindir}/gtk2hs-demo-combodemo
-%{_bindir}/gtk2hs-demo-fastDraw
-%{_bindir}/gtk2hs-demo-fonts
-%{_bindir}/gtk2hs-demo-helloworld
-%{_bindir}/gtk2hs-demo-layout
-%{_bindir}/gtk2hs-demo-menudemo
-%{_bindir}/gtk2hs-demo-notebook
-%{_bindir}/gtk2hs-demo-overlay
-%{_bindir}/gtk2hs-demo-progress
-%{_bindir}/gtk2hs-demo-progressThreadedRTS
-%{_bindir}/gtk2hs-demo-statusIcon
 %dir %{_datadir}/%{pkg_name}-%{version}
 %dir %{_datadir}/%{pkg_name}-%{version}/actionMenu
 %dir %{_datadir}/%{pkg_name}-%{version}/buttonbox

