Index: ghc-hslua.spec
===================================================================
--- ghc-hslua.spec	(revision 2)
+++ ghc-hslua.spec	(working copy)
@@ -40,6 +40,7 @@
 BuildRequires:  ghc-text-devel
 %endif
 # End cabal-rpm deps
+BuildRequires:  lua51-devel
 
 %description
 The Scripting.Lua module is a wrapper of Lua language interpreter as described
@@ -57,6 +58,7 @@
 Requires:       ghc-compiler = %{ghc_version}
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
+Requires:       lua51-devel
 
 %description devel
 This package provides the Haskell %{pkg_name} library development files.
@@ -66,6 +68,7 @@
 
 
 %build
+%define cabal_configure_options -fsystem-lua
 %ghc_lib_build
 
 

