Index: ghc-hslua.spec
===================================================================
--- ghc-hslua.spec	(revision 10)
+++ ghc-hslua.spec	(working copy)
@@ -29,6 +29,6 @@
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-bytestring-devel
 BuildRequires:  ghc-rpm-macros
-BuildRequires:  lua-devel
+BuildRequires:  lua51-devel
 %if %{with tests}
 BuildRequires:  ghc-HUnit-devel
@@ -54,7 +54,7 @@
 Group:          Development/Libraries/Other
 Requires:       %{name} = %{version}-%{release}
 Requires:       ghc-compiler = %{ghc_version}
-Requires:       lua-devel
+Requires:       lua51-devel
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
 

