Index: ghc-hslua.spec
===================================================================
--- ghc-hslua.spec	(revision 10)
+++ ghc-hslua.spec	(working copy)
@@ -29,6 +29,6 @@
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-bytestring-devel
 BuildRequires:  ghc-rpm-macros
-BuildRequires:  lua-devel
+BuildRequires:  lua51-devel
 %if %{with tests}
 BuildRequires:  ghc-HUnit-devel
@@ -54,6 +54,6 @@
 Group:          Development/Libraries/Haskell
 Requires:       %{name} = %{version}-%{release}
 Requires:       ghc-compiler = %{ghc_version}
-Requires:       lua-devel
+Requires:       lua51-devel
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
@@ -66,3 +66,3 @@
 %build
-%define cabal_configure_options -fsystem-lua
+%define cabal_configure_options -fsystem-lua --extra-include-dirs=/usr/include/lua5.1
 %ghc_lib_build

