Index: ghc-proto-lens-protobuf-types.spec
===================================================================
--- ghc-proto-lens-protobuf-types.spec	(revision 1)
+++ ghc-proto-lens-protobuf-types.spec	(working copy)
@@ -64,6 +64,13 @@
 %files -f %{name}.files
 %defattr(-,root,root,-)
 %doc LICENSE
+%dir /usr/share/proto-lens-protobuf-types-0.2.1.0
+%dir /usr/share/proto-lens-protobuf-types-0.2.1.0/proto-lens-imports
+%dir /usr/share/proto-lens-protobuf-types-0.2.1.0/proto-lens-imports/google
+%dir /usr/share/proto-lens-protobuf-types-0.2.1.0/proto-lens-imports/google/protobuf
+/usr/share/proto-lens-protobuf-types-0.2.1.0/proto-lens-imports/google/protobuf/any.proto
+/usr/share/proto-lens-protobuf-types-0.2.1.0/proto-lens-imports/google/protobuf/duration.proto
+/usr/share/proto-lens-protobuf-types-0.2.1.0/proto-lens-imports/google/protobuf/wrappers.proto
 
 %files devel -f %{name}-devel.files
 %defattr(-,root,root,-)

