Index: ghc-th-lift.spec
===================================================================
--- ghc-th-lift.spec	(revision 1)
+++ ghc-th-lift.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        0.7.6
 Release:        0
 Summary:        Derive Template Haskell's Lift class for datatypes
-License:        BSD-3-Clause
+License:        BSD-3-Clause OR GPL-2.0
 Group:          Development/Languages/Haskell
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

