Index: ghc-hyphenation.spec
===================================================================
--- ghc-hyphenation.spec	(revision 9)
+++ ghc-hyphenation.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        0.7
 Release:        0
 Summary:        Configurable Knuth-Liang hyphenation
-License:        BSD-3-Clause
+License:        BSD-2-Clause
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

