Index: ghc-folds.spec
===================================================================
--- ghc-folds.spec	(revision 2)
+++ ghc-folds.spec	(working copy)
@@ -22,6 +22,6 @@
 Version:        0.7.1
 Release:        0
 Summary:        Beautiful Folding
-License:        BSD-3-Clause
+License:        BSD-2-Clause
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

