Index: ghc-regex-compat.spec
===================================================================
--- ghc-regex-compat.spec	(revision 1)
+++ ghc-regex-compat.spec	(working copy)
@@ -25,6 +25,7 @@
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
 Source1:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/revision/1.cabal#/%{pkg_name}.cabal
+Patch1:         https://gitlab.haskell.org/ghc/head.hackage/-/blob/master/patches/regex-compat-0.95.2.0.patch
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-array-devel
 BuildRequires:  ghc-regex-base-devel
@@ -48,7 +49,8 @@
 This package provides the Haskell %{pkg_name} library development files.
 
 %prep
-%autosetup -n %{pkg_name}-%{version}
+%autosetup -n %{pkg_name}-%{version} -p1
 cp -p %{SOURCE1} %{pkg_name}.cabal
+cabal-tweak-dep-ver base '< 4.15' '<5'
 
 %build
