Index: ghc-glazier.spec
===================================================================
--- ghc-glazier.spec	(revision 1)
+++ ghc-glazier.spec	(working copy)
@@ -36,7 +36,11 @@
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 
 %description
-Composable widgets framework.
+Functional version of Elm Action/Model/View/Update architecture, but
+additionally powered by Haskell typeclasses, monad transformers, and lens. In
+Haskell, we can use lens, prisms and typeclasses to embed a smaller widget
+within a larger widget; without modifying any existing widget code or manual
+"lifting of state".
 
 %package devel
 Summary:        Haskell %{pkg_name} library development files

