Index: ShellCheck.spec
===================================================================
--- ShellCheck.spec	(revision 12)
+++ ShellCheck.spec	(working copy)
@@ -41,6 +41,7 @@
 BuildRequires:  ghc-process-devel
 BuildRequires:  ghc-regex-tdfa-devel
 BuildRequires:  ghc-rpm-macros
+BuildRequires:  pandoc
 # GHC dont support i586
 %ifarch %{ix86}
 ExclusiveArch:  i586 i686
@@ -85,6 +86,8 @@
 %install
 %ghc_lib_install
 %ghc_fix_rpath %{pkg_name}-%{version}
+pandoc -s -f markdown-smart -t man shellcheck.1.md -o shellcheck.1
+install -Dpm 0644 shellcheck.1 %{buildroot}%{_mandir}/man1/shellcheck.1
 
 %check
 %cabal_test
@@ -99,6 +102,7 @@
 %license LICENSE
 %doc README.md
 %{_bindir}/shellcheck
+%{_mandir}/man1/shellcheck.1*
 
 %files -n ghc-%{name} -f ghc-%{name}.files
 %license LICENSE
