Index: ShellCheck.spec
===================================================================
--- ShellCheck.spec	(revision 16)
+++ ShellCheck.spec	(working copy)
@@ -41,6 +41,7 @@
 BuildRequires:  ghc-process-devel
 BuildRequires:  ghc-regex-tdfa-devel
 BuildRequires:  ghc-rpm-macros
 ExcludeArch:    %{ix86}
+BuildRequires:  pandoc
 
 %description
@@ -80,6 +81,8 @@
 %install
 %ghc_lib_install
 %ghc_fix_rpath %{pkg_name}-%{version}
+pandoc -s -f markdown-smart -t man shellcheck.1.md -o shellcheck.1
+install -Dpm 0644 shellcheck.1 %{buildroot}%{_mandir}/man1/shellcheck.1
 
 %check
 %cabal_test
@@ -94,6 +97,7 @@
 %license LICENSE
 %doc README.md
 %{_bindir}/shellcheck
+%{_mandir}/man1/shellcheck.1%{?ext_man}
 
 %files -n ghc-%{name} -f ghc-%{name}.files
 %license LICENSE
