Index: ShellCheck.spec
===================================================================
--- ShellCheck.spec	(revision 1)
+++ ShellCheck.spec	(working copy)
@@ -25,7 +25,7 @@
 License:        GPL-3.0-or-later
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{name}
-Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
+Source0:        https://github.com/koalaman/shellcheck/archive/v0.4.7-106-g235bf66.tar.gz#/%{name}-%{version}.tar.gz
 Source1:        https://hackage.haskell.org/package/%{name}-%{version}/revision/1.cabal#/%{name}.cabal
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-QuickCheck-devel
@@ -37,6 +37,7 @@
 BuildRequires:  ghc-process-devel
 BuildRequires:  ghc-regex-tdfa-devel
 BuildRequires:  ghc-rpm-macros
+BuildRequires:  pandoc
 
 %description
 The goals of ShellCheck are:
@@ -69,14 +70,14 @@
 This package provides the Haskell %{name} library development files.
 
 %prep
-%setup -q
-cp -p %{SOURCE1} %{name}.cabal
+%setup -q -n shellcheck-235bf6605fed31c90b17f3f76a30945ef344d015
 
 %build
 %ghc_lib_build
 
 %install
 %ghc_lib_install
+pandoc -s -t man shellcheck.1.md -o shellcheck.1
 install -Dpm 0644 shellcheck.1 %{buildroot}%{_mandir}/man1/shellcheck.1
 
 %check

