Index: ghc-yes-precure5-command.spec
===================================================================
--- ghc-yes-precure5-command.spec	(revision 3)
+++ ghc-yes-precure5-command.spec	(working copy)
@@ -70,7 +70,8 @@
 
 %install
 %ghc_lib_install
-%ghc_fix_rpath %{pkg_name}-%{version}
+mv %{buildroot}%{_bindir}/yes %{buildroot}%{_bindir}/yes-precure5
+%ghc_fix_rpath %{pkg_name}-%{version}
 
 
 %check
@@ -89,5 +89,5 @@
 %files -f %{name}.files
 %doc LICENSE
-%{_bindir}/yes
+%{_bindir}/yes-precure5
 
 %files devel -f %{name}-devel.files

