Index: ghc-uri-encode.spec
===================================================================
--- ghc-uri-encode.spec	(revision 5)
+++ ghc-uri-encode.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        1.5.0.4
 Release:        0
 Summary:        Unicode aware uri-encoding
-License:        Unknown
+License:        BSD-3-Clause
 Group:          Development/Languages/Other
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

