Index: cabal-rpm.spec
===================================================================
--- cabal-rpm.spec	(revision 8)
+++ cabal-rpm.spec	(working copy)
@@ -49,6 +49,7 @@
 
 %build
 %ghc_bin_build
+install -p -m 0644 -D man/cblrpm.1 %{buildroot}%{_mandir}/man1/cblrpm.1
 
 %install
 %ghc_bin_install
@@ -58,5 +59,6 @@
 %doc COPYING
 %doc ChangeLog README.md
 %{_bindir}/cblrpm
+%{_mandir}/man1/cblrpm.1*
 
 %changelog

