Index: ghc-ihaskell.spec
===================================================================
--- ghc-ihaskell.spec	(revision 2)
+++ ghc-ihaskell.spec	(working copy)
@@ -31,7 +31,6 @@
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-aeson-devel
 BuildRequires:  ghc-base64-bytestring-devel
-BuildRequires:  ghc-bin-package-db-devel
 BuildRequires:  ghc-bytestring-devel
 BuildRequires:  ghc-cereal-devel
 BuildRequires:  ghc-cmdargs-devel

