Index: ghc-gogol-drive.spec
===================================================================
--- ghc-gogol-drive.spec	(revision 5)
+++ ghc-gogol-drive.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.1.0
 Release:        0
 Summary:        Google Drive SDK
-License:        Unknown
+License:        MPL-2.0
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

