Index: Allure.spec
===================================================================
--- Allure.spec	(revision 1)
+++ Allure.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.8.1.2
 Release:        0
 Summary:        Near-future Sci-Fi roguelike and tactical squad game
-License:        Unknown
+License:        AGPL-3.0+ AND GPL-2.0-only AND OFL-1.1
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz

