Index: ghc-persistent-sqlite.spec
===================================================================
--- ghc-persistent-sqlite.spec	(revision 2)
+++ ghc-persistent-sqlite.spec	(working copy)
@@ -48,5 +48,6 @@
 BuildRequires:  ghc-persistent-template-devel
 %endif
+BuildRequires:  sqlite3-devel
 
 %description
 This package includes a thin sqlite3 wrapper based on the direct-sqlite
@@ -63,5 +64,6 @@
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
+Requires:       sqlite3-devel
 
 %description devel
 This package provides the Haskell %{pkg_name} library development
@@ -70,7 +72,8 @@
 
 %prep
 %setup -q -n %{pkg_name}-%{version}
 cp -p %{SOURCE1} %{pkg_name}.cabal
+cabal-tweak-flag systemlib True
 
 %build
 %ghc_lib_build

