Index: ghc-gio.spec
===================================================================
--- ghc-gio.spec	(revision 9)
+++ ghc-gio.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.13.1.1
 Release:        0
 Summary:        Binding to GIO
-License:        LGPL-2.1+
+License:        LGPL-2.1+ AND LGPL-3.0+
 Group:          Development/Languages/Haskell
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

