Index: ghc-amazonka-pinpoint.spec
===================================================================
--- ghc-amazonka-pinpoint.spec	(revision 1)
+++ ghc-amazonka-pinpoint.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        1.4.5
 Release:        0
 Summary:        Amazon Pinpoint SDK
-License:        Unknown
+License:        MPL-2.0
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

