Index: ghc-snappy.spec
===================================================================
--- ghc-snappy.spec	(revision 3)
+++ ghc-snappy.spec	(working copy)
@@ -26,6 +26,7 @@
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
+BuildRequires:  gcc-c++
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-bytestring-devel
 BuildRequires:  ghc-rpm-macros

