Index: ghc-unique.spec
===================================================================
--- ghc-unique.spec	(revision 3)
+++ ghc-unique.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0
 Release:        0
 Summary:        Fully concurrent unique identifiers
-License:        BSD-3-Clause
+License:        BSD-2-Clause
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

