Index: ghc-bitcoin-payment-channel.spec
===================================================================
--- ghc-bitcoin-payment-channel.spec	(revision 1)
+++ ghc-bitcoin-payment-channel.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        1.0.1.0
 Release:        0
 Summary:        Instant, two-party Bitcoin payments
-License:        Unknown
+License:        CC0-1.0
 Group:          Development/Languages/Other
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

