Index: ghc-semver.spec
===================================================================
--- ghc-semver.spec	(revision 5)
+++ ghc-semver.spec	(working copy)
@@ -22,6 +22,6 @@
 Version:        0.3.3.1
 Release:        0
 Summary:        Representation, manipulation, and de/serialisation of Semantic Versions
-License:        Unknown
+License:        MPL-2.0
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

