Index: ghc-amazonka-ssm.spec
===================================================================
--- ghc-amazonka-ssm.spec	(revision 3)
+++ ghc-amazonka-ssm.spec	(working copy)
@@ -22,4 +22,4 @@
-License:        Unknown
+License:        MPL-2.0
 Group:          Development/Languages/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

