Index: ghc-log-domain.spec
===================================================================
--- ghc-log-domain.spec	(revision 2)
+++ ghc-log-domain.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        0.11
 Release:        0
 Summary:        Log-domain arithmetic
-License:        BSD-3-Clause
+License:        BSD-2-Clause
 Group:          Development/Languages/Haskell
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

