Index: ghc-yesod-static-angular.spec
===================================================================
--- ghc-yesod-static-angular.spec	(revision 5)
+++ ghc-yesod-static-angular.spec	(working copy)
@@ -26,7 +26,6 @@
 Group:          Development/Languages/Other
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
-BuildRequires:  chrpath
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-aeson-devel
 BuildRequires:  ghc-blaze-builder-devel
@@ -77,13 +76,12 @@
 
 
 %build
-%define cabal_configure_options -fexample
+%define cabal_configure_options -f-example
 %ghc_lib_build
 
 
 %install
 %ghc_lib_install
-%ghc_fix_dynamic_rpath example-dev example-production
 
 
 %check
@@ -99,8 +97,6 @@
 %files -f %{name}.files
 %defattr(-,root,root,-)
 %doc LICENSE
-%{_bindir}/example-dev
-%{_bindir}/example-production
 
 %files devel -f %{name}-devel.files
 %defattr(-,root,root,-)

