Index: cpphs.spec
===================================================================
--- cpphs.spec	(revision 9)
+++ cpphs.spec	(working copy)
@@ -21,6 +21,6 @@
 Version:        1.20.1
 Release:        0
 Summary:        A liberalised re-implementation of cpp, the C pre-processor
-License:        LGPL-2.0-or-later
+License:        GPL-2.0-only AND LGPL-2.1-only
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz
@@ -45,5 +45,6 @@
 
 %package -n ghc-%{name}
 Summary:        Haskell %{name} library
+License:        LGPL-2.1-only
 
 %description -n ghc-%{name}
@@ -52,5 +53,6 @@
 
 %package -n ghc-%{name}-devel
 Summary:        Haskell %{name} library development files
+License:        LGPL-2.1-only
 Requires:       ghc-%{name} = %{version}-%{release}
 Requires:       ghc-compiler = %{ghc_version}

