Index: ghc-mustache.spec
===================================================================
--- ghc-mustache.spec	(revision 4)
+++ ghc-mustache.spec	(working copy)
@@ -25,6 +25,7 @@
 License:        BSD-3-Clause
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
+Patch0:         mustache-2.3.1.patch
 BuildRequires:  chrpath
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-aeson-devel
@@ -75,7 +76,7 @@
 This package provides the Haskell %{pkg_name} library development files.
 
 %prep
-%autosetup -n %{pkg_name}-%{version}
+%autosetup -p1 -n %{pkg_name}-%{version}
 
 %build
 %ghc_lib_build
