Index: ghc-cryptonite-openssl.spec
===================================================================
--- ghc-cryptonite-openssl.spec	(revision 1)
+++ ghc-cryptonite-openssl.spec	(working copy)
@@ -41,7 +41,7 @@
 %endif
 
 %description
-Cryptography.
+Support for OpenSSL based crypto operations in Haskell, as a bolt-in to 'cryptonite'.
 
 %package devel
 Summary:        Haskell %{pkg_name} library development files

