Index: ghc-filelock.spec
===================================================================
--- ghc-filelock.spec	(revision 2)
+++ ghc-filelock.spec	(working copy)
@@ -21,6 +21,6 @@
 Version:        0.1.0.1
 Release:        0
 Summary:        Portable interface to file locking (flock / LockFileEx)
-License:        SUSE-Public-Domain
+License:        CC0-1.0
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

