Index: ghc-postgresql-libpq.spec
===================================================================
--- ghc-postgresql-libpq.spec	(revision 1)
+++ ghc-postgresql-libpq.spec	(working copy)
@@ -31,6 +31,7 @@
 BuildRequires:  ghc-rpm-macros
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 # End cabal-rpm deps
+BuildRequires:  postgresql-devel
 
 %description
 This is a binding to libpq: the C application programmer's interface to

