Index: ghc-colour.spec
===================================================================
--- ghc-colour.spec	(revision 3)
+++ ghc-colour.spec	(working copy)
@@ -55,6 +55,8 @@
 
 %install
 %ghc_lib_install
+rm %{buildroot}/%{_datadir}/%{pkg_name}-%{version}/README
+rm %{buildroot}/%{_datadir}/%{pkg_name}-%{version}/CHANGELOG
 
 
 %post devel
@@ -66,11 +68,9 @@
 %files -f %{name}.files
 %defattr(-,root,root,-)
 %doc LICENSE
-%dir %{_datadir}/%{pkg_name}-%{version}
-%{_datadir}/%{pkg_name}-%{version}/README
-%{_datadir}/%{pkg_name}-%{version}/CHANGELOG
 
 %files devel -f %{name}-devel.files
 %defattr(-,root,root,-)
+%doc README
 
 %changelog

