Index: find-clumpiness.spec
===================================================================
--- find-clumpiness.spec	(revision 5)
+++ find-clumpiness.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.2.0.1
 Release:        0
 Summary:        Find the clumpiness of labels in a tree
-License:        GPL-2.0+
+License:        GPL-3.0
 Group:          Development/Languages/Haskell
 URL:            https://hackage.haskell.org/package/%{name}
 Source0:        https://hackage.haskell.org/package/%{name}-%{version}/%{name}-%{version}.tar.gz

