Index: ghc-fixed.spec
===================================================================
--- ghc-fixed.spec	(revision 2)
+++ ghc-fixed.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.2.1.1
 Release:        0
 Summary:        Signed 15.16 precision fixed point arithmetic
-License:        BSD-3-Clause
+License:        BSD-2-Clause
 Group:          System/Libraries
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

