Index: ghc-hsass.spec
===================================================================
--- ghc-hsass.spec	(revision 2)
+++ ghc-hsass.spec	(working copy)
@@ -26,6 +26,7 @@
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
 Source1:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/revision/1.cabal#/%{pkg_name}.cabal
+BuildRequires:  gcc-c++
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-bytestring-devel

