Index: ghc-wai-middleware-verbs.spec
===================================================================
--- ghc-wai-middleware-verbs.spec	(revision 1)
+++ ghc-wai-middleware-verbs.spec	(working copy)
@@ -42,5 +42,8 @@
 BuildRequires:  ghc-wai-devel
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+BuildRequires:  ghc-wai-middleware-content-type-devel
+BuildRequires:  ghc-wai-transformers-devel
+BuildRequires:  ghc-warp-devel
 
 %description
 Route different middleware responses based on the incoming HTTP verb.

