Index: alex.spec
===================================================================
--- alex.spec	(revision 5)
+++ alex.spec	(working copy)
@@ -15,13 +15,9 @@
 # Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
+# Tests currently don't work on ghc-9.0.x.
+%bcond_with tests
 
-# Disable tests on aarch64. See: https://github.com/simonmar/alex/issues/130
-%ifarch aarch64
-%bcond_with tests
-%else
-%bcond_without tests
-%endif
 Name:           alex
 Version:        3.2.5
 Release:        0
