Index: ghc-libyaml.spec
===================================================================
--- ghc-libyaml.spec	(revision 1)
+++ ghc-libyaml.spec	(working copy)
@@ -32,8 +32,8 @@
 BuildRequires:  ghc-bytestring-prof
 BuildRequires:  ghc-conduit-devel
 BuildRequires:  ghc-conduit-prof
-BuildRequires:  ghc-libyaml-clib-devel
-BuildRequires:  ghc-libyaml-clib-prof
+BuildRequires:  libyaml-devel
+Requires:       libyaml-devel
 BuildRequires:  ghc-resourcet-devel
 BuildRequires:  ghc-resourcet-prof
 BuildRequires:  ghc-rpm-macros
