Index: ghc-postgresql-simple.spec
===================================================================
--- ghc-postgresql-simple.spec	(revision 15)
+++ ghc-postgresql-simple.spec	(working copy)
@@ -27,5 +27,6 @@
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
+Patch0:         https://github.com/haskellari/postgresql-simple/pull/110.patch#/dont-depend-on-obsolete-bytestring-builder.patch
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-Only-devel
 BuildRequires:  ghc-Only-prof
@@ -36,8 +37,6 @@
 BuildRequires:  ghc-attoparsec-prof
 BuildRequires:  ghc-base-devel
 BuildRequires:  ghc-base-prof
-BuildRequires:  ghc-bytestring-builder-devel
-BuildRequires:  ghc-bytestring-builder-prof
 BuildRequires:  ghc-bytestring-devel
 BuildRequires:  ghc-bytestring-prof
 BuildRequires:  ghc-case-insensitive-devel
@@ -114,7 +113,8 @@
 This package provides the Haskell %{pkg_name} profiling library.
 
 %prep
-%autosetup -n %{pkg_name}-%{version}
+%setup -n %{pkg_name}-%{version}
+%patch0 -p1
 
 %build
 %ghc_lib_build
