Index: ghc-ViennaRNAParser.spec
===================================================================
--- ghc-ViennaRNAParser.spec	(revision 2)
+++ ghc-ViennaRNAParser.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        1.2.9
 Release:        0
 Summary:        Libary for parsing ViennaRNA package output
-License:        GPL-3.0+
+License:        GPL-3.0
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

