Index: ghc-yi-rope.spec
===================================================================
--- ghc-yi-rope.spec	(revision 4)
+++ ghc-yi-rope.spec	(working copy)
@@ -56,6 +56,7 @@
 Requires:       ghc-compiler = %{ghc_version}
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
+Requires:       libstdc++-devel
 
 %description devel
 This package provides the Haskell %{pkg_name} library development files.

