Index: ghc-http-accept.spec
===================================================================
--- ghc-http-accept.spec	(revision 5)
+++ ghc-http-accept.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.2
 Release:        0
 Summary:        Functions for working with HTTP Accept headers
-License:        Unknown
+License:        MIT
 Group:          Development/Libraries/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

