Index: ghc-email-validate.spec
===================================================================
--- ghc-email-validate.spec	(revision 7)
+++ ghc-email-validate.spec	(working copy)
@@ -54,6 +54,7 @@
 %autosetup -n %{pkg_name}-%{version}
 sed -i 's/\r//' LICENSE
 cabal-tweak-dep-ver 'attoparsec' '< 0.14' '< 0.15'
+cabal-tweak-dep-ver 'template-haskell' '< 2.17' '< 2.18'
 
 %build
 %ghc_lib_build
