Index: ghc-distribution-opensuse.spec
===================================================================
--- ghc-distribution-opensuse.spec	(revision 7)
+++ ghc-distribution-opensuse.spec	(working copy)
@@ -43,5 +43,6 @@
 BuildRequires:  ghc-text-devel
 BuildRequires:  ghc-time-devel
 BuildRequires:  ghc-turtle-devel
+BuildRequires:  pandoc
 
 %description
@@ -75,10 +76,12 @@
 
 %build
 %ghc_lib_build
+pandoc -s -t man -o guess-changelog.1 guess-changelog.md
 
 %install
 %ghc_lib_install
 %ghc_fix_rpath %{pkg_name}-%{version}
+install -D -m 644 guess-changelog.1 %{buildroot}%{_mandir}/man1/guess-changelog.1
 
 %post devel
 %ghc_pkg_recache
@@ -91,6 +94,7 @@
 
 %files -n guess-changelog
 %{_bindir}/guess-changelog
+%{_mandir}/man1/guess-changelog.1%{?ext_man}
 
 %files devel -f %{name}-devel.files
 
