Index: ghc-servant.spec
===================================================================
--- ghc-servant.spec	(revision 4)
+++ ghc-servant.spec	(working copy)
@@ -25,6 +25,7 @@
 License:        BSD-3-Clause
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
+Patch0:         servant-0.18.2.patch
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-QuickCheck-devel
 BuildRequires:  ghc-aeson-devel
@@ -74,8 +75,9 @@
 This package provides the Haskell %{pkg_name} library development files.
 
 %prep
-%autosetup -n %{pkg_name}-%{version}
+%autosetup -p1 -n %{pkg_name}-%{version}
 cabal-tweak-dep-ver http-api-data "< 0.4.3" "< 0.5"
 cabal-tweak-dep-ver attoparsec '< 0.14' '< 0.15'
+cabal-tweak-dep-ver base "< 4.15" "< 4.16"
 
 %build
 %ghc_lib_build
