Index: darcs.spec
===================================================================
--- darcs.spec	(revision 8)
+++ darcs.spec	(working copy)
@@ -82,6 +82,7 @@
 BuildRequires:  ghc-test-framework-hunit-devel
 BuildRequires:  ghc-test-framework-quickcheck2-devel
 %endif
+Source4:        %{name}.changes
 
 %description
 Darcs is a free, open source revision control system. It is:
@@ -123,5 +124,9 @@
 %prep
 %setup -q
+modified="$(sed -n '/^----/n;s/ - .*$//;p;q' "%{SOURCE4}")"
+DATE="\"$(date -d "${modified}" "+%%b %%e %%Y")\""
+TIME="\"$(date -d "${modified}" "+%%R")\""
+find .  -name '*.[ch]' | xargs sed -i "s/__DATE__/${DATE}/g;s/__TIME__/${TIME}/g"
 
 %build
 %ghc_lib_build

