Index: darcs.spec
===================================================================
--- darcs.spec	(revision 15)
+++ darcs.spec	(working copy)
@@ -84,6 +84,7 @@
 BuildRequires:  ghc-test-framework-quickcheck2-devel
 %endif
 # End cabal-rpm deps
+Source4:        %{name}.changes
 
 %description
 Darcs is a free, open source revision control system. It is:
@@ -126,6 +127,10 @@
 
 %prep
 %setup -q
+modified="$(sed -n '/^----/n;s/ - .*$//;p;q' "%{SOURCE4}")"
+DATE="\"$(date -d "${modified}" "+%%b %%e %%Y")\""
+TIME="\"$(date -d "${modified}" "+%%R")\""
+find .  -name '*.[ch]' | xargs sed -i "s/__DATE__/${DATE}/g;s/__TIME__/${TIME}/g"
 
 
 %build

