Index: ghc-hledger-lib.spec
===================================================================
--- ghc-hledger-lib.spec	(revision 3)
+++ ghc-hledger-lib.spec	(working copy)
@@ -22,7 +22,7 @@
 Version:        1.1
 Release:        0
 Summary:        Core data types, parsers and functionality for the hledger accounting tools
-License:        GPL-1.0+
+License:        GPL-3.0+
 Group:          Development/Languages/Haskell
 URL:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz

