Index: hlint.spec
===================================================================
--- hlint.spec	(revision 4)
+++ hlint.spec	(working copy)
@@ -76,6 +76,7 @@
 
 %install
 %ghc_lib_install
 %ghc_fix_rpath %{pkg_name}-%{version}
+install -D --mode=444 data/hlint.1 %{buildroot}%{_mandir}/man1/hlint.1
 
 %post -n ghc-%{name}-devel
@@ -103,6 +103,7 @@
 %{_datadir}/%{name}-%{version}/hlint.ghci
 %{_datadir}/%{name}-%{version}/HLint_QuickCheck.hs
 %{_datadir}/%{name}-%{version}/HLint_TypeCheck.hs
+%{_mandir}/man1/hlint.1%{ext_man}
 
 %files -n ghc-%{name} -f ghc-%{name}.files
 %defattr(-,root,root,-)

