Index: hlint.spec
===================================================================
--- hlint.spec	(revision 12)
+++ hlint.spec	(working copy)
@@ -80,6 +80,7 @@
 %install
 %ghc_lib_install
 %ghc_fix_rpath %{pkg_name}-%{version}
+install -D --mode=444 data/hlint.1 %{buildroot}%{_mandir}/man1/hlint.1
 
 %post -n ghc-%{name}-devel
 %ghc_pkg_recache
@@ -106,6 +107,7 @@
 %doc LICENSE
 
 %files -n ghc-%{name}-devel -f ghc-%{name}-devel.files
+%{_mandir}/man1/hlint.1%{ext_man}
 %doc CHANGES.txt README.md
 
 %changelog

