Index: ghc-gtk3.spec
===================================================================
--- ghc-gtk3.spec	(revision 25)
+++ ghc-gtk3.spec	(working copy)
@@ -25,6 +25,8 @@
 Group:          System/Libraries
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
+# PATCH-FIX-OPENSUSE work-around-pixel-define.patch -- mimi.vx@gmail.com .. undefine pixel, patch by debian
+Patch0:         work-around-pixel-define.patch
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-array-devel
 BuildRequires:  ghc-bytestring-devel
@@ -65,6 +67,7 @@
 
 %prep
 %setup -q -n %{pkg_name}-%{version}
+%patch0 -p1
 
 
 %build

