Index: ghc-gtk3.spec
===================================================================
--- ghc-gtk3.spec	(revision 4)
+++ ghc-gtk3.spec	(working copy)
@@ -21,7 +21,7 @@
 Version:        0.14.2
 Release:        0
 Summary:        Binding to the Gtk+ 3 graphical user interface library
-License:        LGPL-2+
+License:        LGPL-2.1+
 Group:          System/Libraries
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
@@ -41,6 +41,7 @@
 BuildRequires:  ghc-text-devel
 BuildRequires:  gtk2hs-buildtools
 # End cabal-rpm deps
+BuildRequires:  fdupes
 BuildRequires:  pkgconfig
 BuildRequires:  pkgconfig(gthread-2.0)
 BuildRequires:  pkgconfig(gtk+-3.0)
@@ -73,14 +74,14 @@
 %setup -q -n %{pkg_name}-%{version}
 %patch0 -p1
 
-
 %build
 %ghc_lib_build
 
 
 %install
 %ghc_lib_install
-
+# TODO: repack in all gtk2hs packages examples in right way
+%fdupes %{buildroot}%{_prefix}
 
 %post devel
 %ghc_pkg_recache

