Index: ghc-GLUT.spec
===================================================================
--- ghc-GLUT.spec	(revision 2)
+++ ghc-GLUT.spec	(working copy)
@@ -26,5 +26,6 @@
 Url:            https://hackage.haskell.org/package/%{pkg_name}
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
 BuildRequires:  ghc-Cabal-devel
+BuildRequires:  freeglut-devel
 BuildRequires:  ghc-OpenGL-devel
 BuildRequires:  ghc-StateVar-devel
@@ -46,6 +47,7 @@
 Summary:        Haskell %{pkg_name} library development files
 Group:          Development/Libraries/Other
 Requires:       %{name} = %{version}-%{release}
+Requires:       freeglut-devel
 Requires:       ghc-compiler = %{ghc_version}
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}

