Index: ghc-wai-middleware-verbs.spec
===================================================================
--- ghc-wai-middleware-verbs.spec	(revision 9)
+++ ghc-wai-middleware-verbs.spec	(working copy)
@@ -65,7 +65,7 @@
 
 
 %build
-%define cabal_configure_options -fexample
+%define cabal_configure_options -f-example
 %ghc_lib_build
 
 
@@ -82,7 +82,6 @@
 %files -f %{name}.files
 %defattr(-,root,root,-)
 %doc LICENSE
-%{_bindir}/example
 
 %files devel -f %{name}-devel.files
 %defattr(-,root,root,-)

