Index: ghc-hlibsass.spec
===================================================================
--- ghc-hlibsass.spec	(revision 4)
+++ ghc-hlibsass.spec	(working copy)
@@ -28,6 +28,8 @@
 Source0:        https://hackage.haskell.org/package/%{pkg_name}-%{version}/%{pkg_name}-%{version}.tar.gz
 BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-rpm-macros
+BuildRequires:  libsass-devel
+BuildRequires:  libstdc++6
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 # Begin cabal-rpm deps:
 %if %{with tests}
@@ -47,6 +49,7 @@
 Group:          Development/Libraries/Other
 Requires:       %{name} = %{version}-%{release}
 Requires:       ghc-compiler = %{ghc_version}
+Requires:       libsass-devel
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
 
@@ -55,6 +58,7 @@
 
 %prep
 %setup -q -n %{pkg_name}-%{version}
+%define cabal_configure_options -fexternalLibsass
 
 
 %build

